// @favware/graphql-pokemon/dist/graphql-pokemon.cjs
"use strict";
var __defProp = Object.defineProperty;
var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
var __getOwnPropNames = Object.getOwnPropertyNames;
var __hasOwnProp = Object.prototype.hasOwnProperty;
var __export = (target, all) => {
  for (var name in all)
    __defProp(target, name, { get: all[name], enumerable: true });
};
var __copyProps = (to, from, except, desc) => {
  if (from && typeof from === "object" || typeof from === "function") {
    for (let key of __getOwnPropNames(from))
      if (!__hasOwnProp.call(to, key) && key !== except)
        __defProp(to, key, { get: () => from[key], enumerable: !(desc = __getOwnPropDesc(from, key)) || desc.enumerable });
  }
  return to;
};
var __toCommonJS = (mod) => __copyProps(__defProp({}, "__esModule", { value: true }), mod);
var graphql_pokemon_exports = {};
__export(graphql_pokemon_exports, {
  AbilitiesEnum: () => AbilitiesEnum,
  ItemsEnum: () => ItemsEnum,
  MovesEnum: () => MovesEnum,
  PokemonEnum: () => PokemonEnum,
  TypesEnum: () => TypesEnum
});
module.exports = __toCommonJS(graphql_pokemon_exports);
var AbilitiesEnum = /* @__PURE__ */ ((AbilitiesEnum2) => {
  AbilitiesEnum2["Adaptability"] = "adaptability";
  AbilitiesEnum2["Aerilate"] = "aerilate";
  AbilitiesEnum2["Aftermath"] = "aftermath";
  AbilitiesEnum2["Airlock"] = "airlock";
  AbilitiesEnum2["Analytic"] = "analytic";
  AbilitiesEnum2["Angerpoint"] = "angerpoint";
  AbilitiesEnum2["Anticipation"] = "anticipation";
  AbilitiesEnum2["Arenatrap"] = "arenatrap";
  AbilitiesEnum2["Aromaveil"] = "aromaveil";
  AbilitiesEnum2["Asoneasoneglastrier"] = "asoneasoneglastrier";
  AbilitiesEnum2["Asonespectrier"] = "asonespectrier";
  AbilitiesEnum2["Aurabreak"] = "aurabreak";
  AbilitiesEnum2["Baddreams"] = "baddreams";
  AbilitiesEnum2["Ballfetch"] = "ballfetch";
  AbilitiesEnum2["Battery"] = "battery";
  AbilitiesEnum2["Battlearmor"] = "battlearmor";
  AbilitiesEnum2["Battlebond"] = "battlebond";
  AbilitiesEnum2["Beastboost"] = "beastboost";
  AbilitiesEnum2["Berserk"] = "berserk";
  AbilitiesEnum2["Bigpecks"] = "bigpecks";
  AbilitiesEnum2["Blaze"] = "blaze";
  AbilitiesEnum2["Bulletproof"] = "bulletproof";
  AbilitiesEnum2["Cheekpouch"] = "cheekpouch";
  AbilitiesEnum2["Chillingneigh"] = "chillingneigh";
  AbilitiesEnum2["Chlorophyll"] = "chlorophyll";
  AbilitiesEnum2["Clearbody"] = "clearbody";
  AbilitiesEnum2["Cloudnine"] = "cloudnine";
  AbilitiesEnum2["Colorchange"] = "colorchange";
  AbilitiesEnum2["Comatose"] = "comatose";
  AbilitiesEnum2["Competitive"] = "competitive";
  AbilitiesEnum2["Compoundeyes"] = "compoundeyes";
  AbilitiesEnum2["Contrary"] = "contrary";
  AbilitiesEnum2["Corrosion"] = "corrosion";
  AbilitiesEnum2["Cottondown"] = "cottondown";
  AbilitiesEnum2["Curiousmedicine"] = "curiousmedicine";
  AbilitiesEnum2["Cursedbody"] = "cursedbody";
  AbilitiesEnum2["Cutecharm"] = "cutecharm";
  AbilitiesEnum2["Damp"] = "damp";
  AbilitiesEnum2["Dancer"] = "dancer";
  AbilitiesEnum2["Darkaura"] = "darkaura";
  AbilitiesEnum2["Dauntlessshield"] = "dauntlessshield";
  AbilitiesEnum2["Dazzling"] = "dazzling";
  AbilitiesEnum2["Defeatist"] = "defeatist";
  AbilitiesEnum2["Defiant"] = "defiant";
  AbilitiesEnum2["Deltastream"] = "deltastream";
  AbilitiesEnum2["Desolateland"] = "desolateland";
  AbilitiesEnum2["Disguise"] = "disguise";
  AbilitiesEnum2["Download"] = "download";
  AbilitiesEnum2["Dragonsmaw"] = "dragonsmaw";
  AbilitiesEnum2["Drizzle"] = "drizzle";
  AbilitiesEnum2["Drought"] = "drought";
  AbilitiesEnum2["Dryskin"] = "dryskin";
  AbilitiesEnum2["Earlybird"] = "earlybird";
  AbilitiesEnum2["Effectspore"] = "effectspore";
  AbilitiesEnum2["Electricsurge"] = "electricsurge";
  AbilitiesEnum2["Emergencyexit"] = "emergencyexit";
  AbilitiesEnum2["Fairyaura"] = "fairyaura";
  AbilitiesEnum2["Filter"] = "filter";
  AbilitiesEnum2["Flamebody"] = "flamebody";
  AbilitiesEnum2["Flareboost"] = "flareboost";
  AbilitiesEnum2["Flashfire"] = "flashfire";
  AbilitiesEnum2["Flowergift"] = "flowergift";
  AbilitiesEnum2["Flowerveil"] = "flowerveil";
  AbilitiesEnum2["Fluffy"] = "fluffy";
  AbilitiesEnum2["Forecast"] = "forecast";
  AbilitiesEnum2["Forewarn"] = "forewarn";
  AbilitiesEnum2["Friendguard"] = "friendguard";
  AbilitiesEnum2["Frisk"] = "frisk";
  AbilitiesEnum2["Fullmetalbody"] = "fullmetalbody";
  AbilitiesEnum2["Furcoat"] = "furcoat";
  AbilitiesEnum2["Galewings"] = "galewings";
  AbilitiesEnum2["Galvanize"] = "galvanize";
  AbilitiesEnum2["Gluttony"] = "gluttony";
  AbilitiesEnum2["Gooey"] = "gooey";
  AbilitiesEnum2["Gorillatactics"] = "gorillatactics";
  AbilitiesEnum2["Grasspelt"] = "grasspelt";
  AbilitiesEnum2["Grassysurge"] = "grassysurge";
  AbilitiesEnum2["Grimneigh"] = "grimneigh";
  AbilitiesEnum2["Gulpmissile"] = "gulpmissile";
  AbilitiesEnum2["Guts"] = "guts";
  AbilitiesEnum2["Harvest"] = "harvest";
  AbilitiesEnum2["Healer"] = "healer";
  AbilitiesEnum2["Heatproof"] = "heatproof";
  AbilitiesEnum2["Heavymetal"] = "heavymetal";
  AbilitiesEnum2["Honeygather"] = "honeygather";
  AbilitiesEnum2["Hugepower"] = "hugepower";
  AbilitiesEnum2["Hungerswitch"] = "hungerswitch";
  AbilitiesEnum2["Hustle"] = "hustle";
  AbilitiesEnum2["Hydration"] = "hydration";
  AbilitiesEnum2["Hypercutter"] = "hypercutter";
  AbilitiesEnum2["Icebody"] = "icebody";
  AbilitiesEnum2["Iceface"] = "iceface";
  AbilitiesEnum2["Icescales"] = "icescales";
  AbilitiesEnum2["Illuminate"] = "illuminate";
  AbilitiesEnum2["Illusion"] = "illusion";
  AbilitiesEnum2["Immunity"] = "immunity";
  AbilitiesEnum2["Imposter"] = "imposter";
  AbilitiesEnum2["Infiltrator"] = "infiltrator";
  AbilitiesEnum2["Innardsout"] = "innardsout";
  AbilitiesEnum2["Innerfocus"] = "innerfocus";
  AbilitiesEnum2["Insomnia"] = "insomnia";
  AbilitiesEnum2["Intimidate"] = "intimidate";
  AbilitiesEnum2["Intrepidsword"] = "intrepidsword";
  AbilitiesEnum2["Ironbarbs"] = "ironbarbs";
  AbilitiesEnum2["Ironfist"] = "ironfist";
  AbilitiesEnum2["Justified"] = "justified";
  AbilitiesEnum2["Keeneye"] = "keeneye";
  AbilitiesEnum2["Klutz"] = "klutz";
  AbilitiesEnum2["Leafguard"] = "leafguard";
  AbilitiesEnum2["Levitate"] = "levitate";
  AbilitiesEnum2["Libero"] = "libero";
  AbilitiesEnum2["Lightmetal"] = "lightmetal";
  AbilitiesEnum2["Lightningrod"] = "lightningrod";
  AbilitiesEnum2["Limber"] = "limber";
  AbilitiesEnum2["Liquidooze"] = "liquidooze";
  AbilitiesEnum2["Liquidvoice"] = "liquidvoice";
  AbilitiesEnum2["Longreach"] = "longreach";
  AbilitiesEnum2["Magicbounce"] = "magicbounce";
  AbilitiesEnum2["Magicguard"] = "magicguard";
  AbilitiesEnum2["Magician"] = "magician";
  AbilitiesEnum2["Magmaarmor"] = "magmaarmor";
  AbilitiesEnum2["Magnetpull"] = "magnetpull";
  AbilitiesEnum2["Marvelscale"] = "marvelscale";
  AbilitiesEnum2["Megalauncher"] = "megalauncher";
  AbilitiesEnum2["Merciless"] = "merciless";
  AbilitiesEnum2["Mimicry"] = "mimicry";
  AbilitiesEnum2["Minus"] = "minus";
  AbilitiesEnum2["Mirrorarmor"] = "mirrorarmor";
  AbilitiesEnum2["Mistysurge"] = "mistysurge";
  AbilitiesEnum2["Moldbreaker"] = "moldbreaker";
  AbilitiesEnum2["Moody"] = "moody";
  AbilitiesEnum2["Motordrive"] = "motordrive";
  AbilitiesEnum2["Mountaineer"] = "mountaineer";
  AbilitiesEnum2["Moxie"] = "moxie";
  AbilitiesEnum2["Multiscale"] = "multiscale";
  AbilitiesEnum2["Multitype"] = "multitype";
  AbilitiesEnum2["Mummy"] = "mummy";
  AbilitiesEnum2["Naturalcure"] = "naturalcure";
  AbilitiesEnum2["Neuroforce"] = "neuroforce";
  AbilitiesEnum2["Neutralizinggas"] = "neutralizinggas";
  AbilitiesEnum2["Noability"] = "noability";
  AbilitiesEnum2["Noguard"] = "noguard";
  AbilitiesEnum2["Normalize"] = "normalize";
  AbilitiesEnum2["Oblivious"] = "oblivious";
  AbilitiesEnum2["Overcoat"] = "overcoat";
  AbilitiesEnum2["Overgrow"] = "overgrow";
  AbilitiesEnum2["Owntempo"] = "owntempo";
  AbilitiesEnum2["Parentalbond"] = "parentalbond";
  AbilitiesEnum2["Pastelveil"] = "pastelveil";
  AbilitiesEnum2["Perishbody"] = "perishbody";
  AbilitiesEnum2["Persistent"] = "persistent";
  AbilitiesEnum2["Pickpocket"] = "pickpocket";
  AbilitiesEnum2["Pickup"] = "pickup";
  AbilitiesEnum2["Pixilate"] = "pixilate";
  AbilitiesEnum2["Plus"] = "plus";
  AbilitiesEnum2["Poisonheal"] = "poisonheal";
  AbilitiesEnum2["Poisonpoint"] = "poisonpoint";
  AbilitiesEnum2["Poisontouch"] = "poisontouch";
  AbilitiesEnum2["Powerconstruct"] = "powerconstruct";
  AbilitiesEnum2["Powerofalchemy"] = "powerofalchemy";
  AbilitiesEnum2["Powerspot"] = "powerspot";
  AbilitiesEnum2["Prankster"] = "prankster";
  AbilitiesEnum2["Pressure"] = "pressure";
  AbilitiesEnum2["Primordialsea"] = "primordialsea";
  AbilitiesEnum2["Prismarmor"] = "prismarmor";
  AbilitiesEnum2["Propellertail"] = "propellertail";
  AbilitiesEnum2["Protean"] = "protean";
  AbilitiesEnum2["Psychicsurge"] = "psychicsurge";
  AbilitiesEnum2["Punkrock"] = "punkrock";
  AbilitiesEnum2["Purepower"] = "purepower";
  AbilitiesEnum2["Queenlymajesty"] = "queenlymajesty";
  AbilitiesEnum2["Quickdraw"] = "quickdraw";
  AbilitiesEnum2["Quickfeet"] = "quickfeet";
  AbilitiesEnum2["Raindish"] = "raindish";
  AbilitiesEnum2["Rattled"] = "rattled";
  AbilitiesEnum2["Rebound"] = "rebound";
  AbilitiesEnum2["Receiver"] = "receiver";
  AbilitiesEnum2["Reckless"] = "reckless";
  AbilitiesEnum2["Refrigerate"] = "refrigerate";
  AbilitiesEnum2["Regenerator"] = "regenerator";
  AbilitiesEnum2["Ripen"] = "ripen";
  AbilitiesEnum2["Rivalry"] = "rivalry";
  AbilitiesEnum2["Rkssystem"] = "rkssystem";
  AbilitiesEnum2["Rockhead"] = "rockhead";
  AbilitiesEnum2["Roughskin"] = "roughskin";
  AbilitiesEnum2["Runaway"] = "runaway";
  AbilitiesEnum2["Sandforce"] = "sandforce";
  AbilitiesEnum2["Sandrush"] = "sandrush";
  AbilitiesEnum2["Sandspit"] = "sandspit";
  AbilitiesEnum2["Sandstream"] = "sandstream";
  AbilitiesEnum2["Sandveil"] = "sandveil";
  AbilitiesEnum2["Sapsipper"] = "sapsipper";
  AbilitiesEnum2["Schooling"] = "schooling";
  AbilitiesEnum2["Scrappy"] = "scrappy";
  AbilitiesEnum2["Screencleaner"] = "screencleaner";
  AbilitiesEnum2["Serenegrace"] = "serenegrace";
  AbilitiesEnum2["Shadowshield"] = "shadowshield";
  AbilitiesEnum2["Shadowtag"] = "shadowtag";
  AbilitiesEnum2["Shedskin"] = "shedskin";
  AbilitiesEnum2["Sheerforce"] = "sheerforce";
  AbilitiesEnum2["Shellarmor"] = "shellarmor";
  AbilitiesEnum2["Shielddust"] = "shielddust";
  AbilitiesEnum2["Shieldsdown"] = "shieldsdown";
  AbilitiesEnum2["Simple"] = "simple";
  AbilitiesEnum2["Skilllink"] = "skilllink";
  AbilitiesEnum2["Slowstart"] = "slowstart";
  AbilitiesEnum2["Slushrush"] = "slushrush";
  AbilitiesEnum2["Sniper"] = "sniper";
  AbilitiesEnum2["Snowcloak"] = "snowcloak";
  AbilitiesEnum2["Snowwarning"] = "snowwarning";
  AbilitiesEnum2["Solarpower"] = "solarpower";
  AbilitiesEnum2["Solidrock"] = "solidrock";
  AbilitiesEnum2["Soulheart"] = "soulheart";
  AbilitiesEnum2["Soundproof"] = "soundproof";
  AbilitiesEnum2["Speedboost"] = "speedboost";
  AbilitiesEnum2["Stakeout"] = "stakeout";
  AbilitiesEnum2["Stall"] = "stall";
  AbilitiesEnum2["Stalwart"] = "stalwart";
  AbilitiesEnum2["Stamina"] = "stamina";
  AbilitiesEnum2["Stancechange"] = "stancechange";
  AbilitiesEnum2["Static"] = "static";
  AbilitiesEnum2["Steadfast"] = "steadfast";
  AbilitiesEnum2["Steamengine"] = "steamengine";
  AbilitiesEnum2["Steelworker"] = "steelworker";
  AbilitiesEnum2["Steelyspirit"] = "steelyspirit";
  AbilitiesEnum2["Stench"] = "stench";
  AbilitiesEnum2["Stickyhold"] = "stickyhold";
  AbilitiesEnum2["Stormdrain"] = "stormdrain";
  AbilitiesEnum2["Strongjaw"] = "strongjaw";
  AbilitiesEnum2["Sturdy"] = "sturdy";
  AbilitiesEnum2["Suctioncups"] = "suctioncups";
  AbilitiesEnum2["Superluck"] = "superluck";
  AbilitiesEnum2["Surgesurfer"] = "surgesurfer";
  AbilitiesEnum2["Swarm"] = "swarm";
  AbilitiesEnum2["Sweetveil"] = "sweetveil";
  AbilitiesEnum2["Swiftswim"] = "swiftswim";
  AbilitiesEnum2["Symbiosis"] = "symbiosis";
  AbilitiesEnum2["Synchronize"] = "synchronize";
  AbilitiesEnum2["Tangledfeet"] = "tangledfeet";
  AbilitiesEnum2["Tanglinghair"] = "tanglinghair";
  AbilitiesEnum2["Technician"] = "technician";
  AbilitiesEnum2["Telepathy"] = "telepathy";
  AbilitiesEnum2["Teravolt"] = "teravolt";
  AbilitiesEnum2["Thickfat"] = "thickfat";
  AbilitiesEnum2["Tintedlens"] = "tintedlens";
  AbilitiesEnum2["Torrent"] = "torrent";
  AbilitiesEnum2["Toughclaws"] = "toughclaws";
  AbilitiesEnum2["Toxicboost"] = "toxicboost";
  AbilitiesEnum2["Trace"] = "trace";
  AbilitiesEnum2["Transistor"] = "transistor";
  AbilitiesEnum2["Triage"] = "triage";
  AbilitiesEnum2["Truant"] = "truant";
  AbilitiesEnum2["Turboblaze"] = "turboblaze";
  AbilitiesEnum2["Unaware"] = "unaware";
  AbilitiesEnum2["Unburden"] = "unburden";
  AbilitiesEnum2["Unnerve"] = "unnerve";
  AbilitiesEnum2["Unseenfist"] = "unseenfist";
  AbilitiesEnum2["Victorystar"] = "victorystar";
  AbilitiesEnum2["Vitalspirit"] = "vitalspirit";
  AbilitiesEnum2["Voltabsorb"] = "voltabsorb";
  AbilitiesEnum2["Wanderingspirit"] = "wanderingspirit";
  AbilitiesEnum2["Waterabsorb"] = "waterabsorb";
  AbilitiesEnum2["Waterbubble"] = "waterbubble";
  AbilitiesEnum2["Watercompaction"] = "watercompaction";
  AbilitiesEnum2["Waterveil"] = "waterveil";
  AbilitiesEnum2["Weakarmor"] = "weakarmor";
  AbilitiesEnum2["Whitesmoke"] = "whitesmoke";
  AbilitiesEnum2["Wimpout"] = "wimpout";
  AbilitiesEnum2["Wonderguard"] = "wonderguard";
  AbilitiesEnum2["Wonderskin"] = "wonderskin";
  AbilitiesEnum2["Zenmode"] = "zenmode";
  return AbilitiesEnum2;
})(AbilitiesEnum || {});
var ItemsEnum = /* @__PURE__ */ ((ItemsEnum2) => {
  ItemsEnum2["Abomasite"] = "abomasite";
  ItemsEnum2["Absolite"] = "absolite";
  ItemsEnum2["Absorbbulb"] = "absorbbulb";
  ItemsEnum2["Acrobike"] = "acrobike";
  ItemsEnum2["Adamantorb"] = "adamantorb";
  ItemsEnum2["Adrenalineorb"] = "adrenalineorb";
  ItemsEnum2["Adventureguide"] = "adventureguide";
  ItemsEnum2["Aerodactylite"] = "aerodactylite";
  ItemsEnum2["Aggronite"] = "aggronite";
  ItemsEnum2["Aguavberry"] = "aguavberry";
  ItemsEnum2["Airballoon"] = "airballoon";
  ItemsEnum2["Alakazite"] = "alakazite";
  ItemsEnum2["Aloraichiumz"] = "aloraichiumz";
  ItemsEnum2["Altarianite"] = "altarianite";
  ItemsEnum2["Ampharosite"] = "ampharosite";
  ItemsEnum2["Apicotberry"] = "apicotberry";
  ItemsEnum2["Apricornbox"] = "apricornbox";
  ItemsEnum2["Aquasuit"] = "aquasuit";
  ItemsEnum2["Armorfossil"] = "armorfossil";
  ItemsEnum2["Aspearberry"] = "aspearberry";
  ItemsEnum2["Assaultvest"] = "assaultvest";
  ItemsEnum2["Audinite"] = "audinite";
  ItemsEnum2["Auroraticket"] = "auroraticket";
  ItemsEnum2["Autograph"] = "autograph";
  ItemsEnum2["Azureflute"] = "azureflute";
  ItemsEnum2["Babiriberry"] = "babiriberry";
  ItemsEnum2["Bandautograph"] = "bandautograph";
  ItemsEnum2["Banettite"] = "banettite";
  ItemsEnum2["Basementkey"] = "basementkey";
  ItemsEnum2["Beastball"] = "beastball";
  ItemsEnum2["Beedrillite"] = "beedrillite";
  ItemsEnum2["Belueberry"] = "belueberry";
  ItemsEnum2["Berry"] = "berry";
  ItemsEnum2["Berryjuice"] = "berryjuice";
  ItemsEnum2["Berrypots"] = "berrypots";
  ItemsEnum2["Berrypouch"] = "berrypouch";
  ItemsEnum2["Berrysweet"] = "berrysweet";
  ItemsEnum2["Berserkgene"] = "berserkgene";
  ItemsEnum2["Bicycle"] = "bicycle";
  ItemsEnum2["Bigroot"] = "bigroot";
  ItemsEnum2["Bikevoucher"] = "bikevoucher";
  ItemsEnum2["Bindingband"] = "bindingband";
  ItemsEnum2["Bitterberry"] = "bitterberry";
  ItemsEnum2["Blackbelt"] = "blackbelt";
  ItemsEnum2["Blackglasses"] = "blackglasses";
  ItemsEnum2["Blacksludge"] = "blacksludge";
  ItemsEnum2["Blastoisinite"] = "blastoisinite";
  ItemsEnum2["Blazikenite"] = "blazikenite";
  ItemsEnum2["Bluecard"] = "bluecard";
  ItemsEnum2["Blueorb"] = "blueorb";
  ItemsEnum2["Bluepetal"] = "bluepetal";
  ItemsEnum2["Blukberry"] = "blukberry";
  ItemsEnum2["Blunderpolicy"] = "blunderpolicy";
  ItemsEnum2["Bottlecap"] = "bottlecap";
  ItemsEnum2["Brightpowder"] = "brightpowder";
  ItemsEnum2["Buggem"] = "buggem";
  ItemsEnum2["Buginiumz"] = "buginiumz";
  ItemsEnum2["Bugmemory"] = "bugmemory";
  ItemsEnum2["Burndrive"] = "burndrive";
  ItemsEnum2["Burntberry"] = "burntberry";
  ItemsEnum2["Cameruptite"] = "cameruptite";
  ItemsEnum2["Campinggear"] = "campinggear";
  ItemsEnum2["Cardkey"] = "cardkey";
  ItemsEnum2["Catchingcharm"] = "catchingcharm";
  ItemsEnum2["Cellbattery"] = "cellbattery";
  ItemsEnum2["Charcoal"] = "charcoal";
  ItemsEnum2["Charizarditex"] = "charizarditex";
  ItemsEnum2["Charizarditey"] = "charizarditey";
  ItemsEnum2["Chartiberry"] = "chartiberry";
  ItemsEnum2["Cheriberry"] = "cheriberry";
  ItemsEnum2["Cherishball"] = "cherishball";
  ItemsEnum2["Chestoberry"] = "chestoberry";
  ItemsEnum2["Chilanberry"] = "chilanberry";
  ItemsEnum2["Chilldrive"] = "chilldrive";
  ItemsEnum2["Chippedpot"] = "chippedpot";
  ItemsEnum2["Choiceband"] = "choiceband";
  ItemsEnum2["Choicescarf"] = "choicescarf";
  ItemsEnum2["Choicespecs"] = "choicespecs";
  ItemsEnum2["Chopleberry"] = "chopleberry";
  ItemsEnum2["Clawfossil"] = "clawfossil";
  ItemsEnum2["Clearbell"] = "clearbell";
  ItemsEnum2["Cloversweet"] = "cloversweet";
  ItemsEnum2["Cobaberry"] = "cobaberry";
  ItemsEnum2["Coincase"] = "coincase";
  ItemsEnum2["Colburberry"] = "colburberry";
  ItemsEnum2["Colressmchn"] = "colressmchn";
  ItemsEnum2["Contestcostume"] = "contestcostume";
  ItemsEnum2["Contestpass"] = "contestpass";
  ItemsEnum2["Cornnberry"] = "cornnberry";
  ItemsEnum2["Coupon1"] = "coupon1";
  ItemsEnum2["Coupon2"] = "coupon2";
  ItemsEnum2["Coupon3"] = "coupon3";
  ItemsEnum2["Coverfossil"] = "coverfossil";
  ItemsEnum2["Crackedpot"] = "crackedpot";
  ItemsEnum2["Crucibellite"] = "crucibellite";
  ItemsEnum2["Custapberry"] = "custapberry";
  ItemsEnum2["Damprock"] = "damprock";
  ItemsEnum2["Darkgem"] = "darkgem";
  ItemsEnum2["Darkiniumz"] = "darkiniumz";
  ItemsEnum2["Darkmemory"] = "darkmemory";
  ItemsEnum2["Darkstone"] = "darkstone";
  ItemsEnum2["Dawnstone"] = "dawnstone";
  ItemsEnum2["Decidiumz"] = "decidiumz";
  ItemsEnum2["Deepseascale"] = "deepseascale";
  ItemsEnum2["Deepseatooth"] = "deepseatooth";
  ItemsEnum2["Destinyknot"] = "destinyknot";
  ItemsEnum2["Devongoods"] = "devongoods";
  ItemsEnum2["Devonparts"] = "devonparts";
  ItemsEnum2["Devonscope"] = "devonscope";
  ItemsEnum2["Devonscubagear"] = "devonscubagear";
  ItemsEnum2["Diancite"] = "diancite";
  ItemsEnum2["Diveball"] = "diveball";
  ItemsEnum2["Dnasplicers"] = "dnasplicers";
  ItemsEnum2["Domefossil"] = "domefossil";
  ItemsEnum2["Dousedrive"] = "dousedrive";
  ItemsEnum2["Dowsingmachine"] = "dowsingmachine";
  ItemsEnum2["Dowsingmchn"] = "dowsingmchn";
  ItemsEnum2["Dracoplate"] = "dracoplate";
  ItemsEnum2["Dragonfang"] = "dragonfang";
  ItemsEnum2["Dragongem"] = "dragongem";
  ItemsEnum2["Dragoniumz"] = "dragoniumz";
  ItemsEnum2["Dragonmemory"] = "dragonmemory";
  ItemsEnum2["Dragonscale"] = "dragonscale";
  ItemsEnum2["Dragonskull"] = "dragonskull";
  ItemsEnum2["Dreadplate"] = "dreadplate";
  ItemsEnum2["Dreamball"] = "dreamball";
  ItemsEnum2["Droppeditem"] = "droppeditem";
  ItemsEnum2["Dubiousdisc"] = "dubiousdisc";
  ItemsEnum2["Durinberry"] = "durinberry";
  ItemsEnum2["Duskball"] = "duskball";
  ItemsEnum2["Duskstone"] = "duskstone";
  ItemsEnum2["Dynamaxband"] = "dynamaxband";
  ItemsEnum2["Earthplate"] = "earthplate";
  ItemsEnum2["Eeviumz"] = "eeviumz";
  ItemsEnum2["Eggcard"] = "eggcard";
  ItemsEnum2["Ejectbutton"] = "ejectbutton";
  ItemsEnum2["Ejectpack"] = "ejectpack";
  ItemsEnum2["Electirizer"] = "electirizer";
  ItemsEnum2["Electricgem"] = "electricgem";
  ItemsEnum2["Electricmemory"] = "electricmemory";
  ItemsEnum2["Electricseed"] = "electricseed";
  ItemsEnum2["Electriumz"] = "electriumz";
  ItemsEnum2["Elevatorkey"] = "elevatorkey";
  ItemsEnum2["Endorsement"] = "endorsement";
  ItemsEnum2["Energypowder"] = "energypowder";
  ItemsEnum2["Enigmaberry"] = "enigmaberry";
  ItemsEnum2["Enigmastone"] = "enigmastone";
  ItemsEnum2["Enigmaticcard"] = "enigmaticcard";
  ItemsEnum2["Eonflute"] = "eonflute";
  ItemsEnum2["Eonticket"] = "eonticket";
  ItemsEnum2["Escaperope"] = "escaperope";
  ItemsEnum2["Eviolite"] = "eviolite";
  ItemsEnum2["Expertbelt"] = "expertbelt";
  ItemsEnum2["Explorerkit"] = "explorerkit";
  ItemsEnum2["Expshare"] = "expshare";
  ItemsEnum2["Fairiumz"] = "fairiumz";
  ItemsEnum2["Fairygem"] = "fairygem";
  ItemsEnum2["Fairymemory"] = "fairymemory";
  ItemsEnum2["Famechecker"] = "famechecker";
  ItemsEnum2["Fashioncase"] = "fashioncase";
  ItemsEnum2["Fastball"] = "fastball";
  ItemsEnum2["Fightinggem"] = "fightinggem";
  ItemsEnum2["Fightingmemory"] = "fightingmemory";
  ItemsEnum2["Fightiniumz"] = "fightiniumz";
  ItemsEnum2["Figyberry"] = "figyberry";
  ItemsEnum2["Firegem"] = "firegem";
  ItemsEnum2["Firememory"] = "firememory";
  ItemsEnum2["Firestone"] = "firestone";
  ItemsEnum2["Firiumz"] = "firiumz";
  ItemsEnum2["Fishingrod"] = "fishingrod";
  ItemsEnum2["Fistplate"] = "fistplate";
  ItemsEnum2["Flameorb"] = "flameorb";
  ItemsEnum2["Flameplate"] = "flameplate";
  ItemsEnum2["Floatstone"] = "floatstone";
  ItemsEnum2["Flowersweet"] = "flowersweet";
  ItemsEnum2["Flyinggem"] = "flyinggem";
  ItemsEnum2["Flyingmemory"] = "flyingmemory";
  ItemsEnum2["Flyiniumz"] = "flyiniumz";
  ItemsEnum2["Focusband"] = "focusband";
  ItemsEnum2["Focussash"] = "focussash";
  ItemsEnum2["Foragebag"] = "foragebag";
  ItemsEnum2["Fossilizedbird"] = "fossilizedbird";
  ItemsEnum2["Fossilizeddino"] = "fossilizeddino";
  ItemsEnum2["Fossilizeddrake"] = "fossilizeddrake";
  ItemsEnum2["Fossilizedfish"] = "fossilizedfish";
  ItemsEnum2["Friendball"] = "friendball";
  ItemsEnum2["Fullincense"] = "fullincense";
  ItemsEnum2["Fullrestore"] = "fullrestore";
  ItemsEnum2["Galactickey"] = "galactickey";
  ItemsEnum2["Galaricacuff"] = "galaricacuff";
  ItemsEnum2["Galaricawreath"] = "galaricawreath";
  ItemsEnum2["Galladite"] = "galladite";
  ItemsEnum2["Ganlonberry"] = "ganlonberry";
  ItemsEnum2["Garchompite"] = "garchompite";
  ItemsEnum2["Gardevoirite"] = "gardevoirite";
  ItemsEnum2["Gbsounds"] = "gbsounds";
  ItemsEnum2["Gengarite"] = "gengarite";
  ItemsEnum2["Ghostgem"] = "ghostgem";
  ItemsEnum2["Ghostiumz"] = "ghostiumz";
  ItemsEnum2["Ghostmemory"] = "ghostmemory";
  ItemsEnum2["Glalitite"] = "glalitite";
  ItemsEnum2["Godstone"] = "godstone";
  ItemsEnum2["Gogoggles"] = "gogoggles";
  ItemsEnum2["Goldberry"] = "goldberry";
  ItemsEnum2["Goldbottlecap"] = "goldbottlecap";
  ItemsEnum2["Goldteeth"] = "goldteeth";
  ItemsEnum2["Goodrod"] = "goodrod";
  ItemsEnum2["Gracidea"] = "gracidea";
  ItemsEnum2["Gram1"] = "gram1";
  ItemsEnum2["Gram2"] = "gram2";
  ItemsEnum2["Gram3"] = "gram3";
  ItemsEnum2["Grassgem"] = "grassgem";
  ItemsEnum2["Grassiumz"] = "grassiumz";
  ItemsEnum2["Grassmemory"] = "grassmemory";
  ItemsEnum2["Grassyseed"] = "grassyseed";
  ItemsEnum2["Greatball"] = "greatball";
  ItemsEnum2["Greenpetal"] = "greenpetal";
  ItemsEnum2["Grepaberry"] = "grepaberry";
  ItemsEnum2["Gripclaw"] = "gripclaw";
  ItemsEnum2["Griseousorb"] = "griseousorb";
  ItemsEnum2["Groundgem"] = "groundgem";
  ItemsEnum2["Groundiumz"] = "groundiumz";
  ItemsEnum2["Groundmemory"] = "groundmemory";
  ItemsEnum2["Grubbyhanky"] = "grubbyhanky";
  ItemsEnum2["Gsball"] = "gsball";
  ItemsEnum2["Gyaradosite"] = "gyaradosite";
  ItemsEnum2["Habanberry"] = "habanberry";
  ItemsEnum2["Hardstone"] = "hardstone";
  ItemsEnum2["Healball"] = "healball";
  ItemsEnum2["Heatrock"] = "heatrock";
  ItemsEnum2["Heavyball"] = "heavyball";
  ItemsEnum2["Heavydutyboots"] = "heavydutyboots";
  ItemsEnum2["Helixfossil"] = "helixfossil";
  ItemsEnum2["Heracronite"] = "heracronite";
  ItemsEnum2["Hitechearbuds"] = "hitechearbuds";
  ItemsEnum2["Holocaster"] = "holocaster";
  ItemsEnum2["Hondewberry"] = "hondewberry";
  ItemsEnum2["Honorofkalos"] = "honorofkalos";
  ItemsEnum2["Houndoominite"] = "houndoominite";
  ItemsEnum2["Hyperpotion"] = "hyperpotion";
  ItemsEnum2["Iapapaberry"] = "iapapaberry";
  ItemsEnum2["Iceberry"] = "iceberry";
  ItemsEnum2["Icegem"] = "icegem";
  ItemsEnum2["Icememory"] = "icememory";
  ItemsEnum2["Icestone"] = "icestone";
  ItemsEnum2["Icicleplate"] = "icicleplate";
  ItemsEnum2["Iciumz"] = "iciumz";
  ItemsEnum2["Icyrock"] = "icyrock";
  ItemsEnum2["Ilimasnormaliumz"] = "ilimasnormaliumz";
  ItemsEnum2["Inciniumz"] = "inciniumz";
  ItemsEnum2["Insectplate"] = "insectplate";
  ItemsEnum2["Intriguingstone"] = "intriguingstone";
  ItemsEnum2["Ironball"] = "ironball";
  ItemsEnum2["Ironplate"] = "ironplate";
  ItemsEnum2["Itemfinder"] = "itemfinder";
  ItemsEnum2["Jabocaberry"] = "jabocaberry";
  ItemsEnum2["Jadeorb"] = "jadeorb";
  ItemsEnum2["Jawfossil"] = "jawfossil";
  ItemsEnum2["Journal"] = "journal";
  ItemsEnum2["Kangaskhanite"] = "kangaskhanite";
  ItemsEnum2["Kasibberry"] = "kasibberry";
  ItemsEnum2["Kebiaberry"] = "kebiaberry";
  ItemsEnum2["Keeberry"] = "keeberry";
  ItemsEnum2["Kelpsyberry"] = "kelpsyberry";
  ItemsEnum2["Keystone"] = "keystone";
  ItemsEnum2["Keytoroom1"] = "keytoroom1";
  ItemsEnum2["Keytoroom2"] = "keytoroom2";
  ItemsEnum2["Keytoroom4"] = "keytoroom4";
  ItemsEnum2["Keytoroom6"] = "keytoroom6";
  ItemsEnum2["Kingsrock"] = "kingsrock";
  ItemsEnum2["Kommoniumz"] = "kommoniumz";
  ItemsEnum2["Laggingtail"] = "laggingtail";
  ItemsEnum2["Lansatberry"] = "lansatberry";
  ItemsEnum2["Latiasite"] = "latiasite";
  ItemsEnum2["Latiosite"] = "latiosite";
  ItemsEnum2["Laxincense"] = "laxincense";
  ItemsEnum2["Leafstone"] = "leafstone";
  ItemsEnum2["Leek"] = "leek";
  ItemsEnum2["Leftovers"] = "leftovers";
  ItemsEnum2["Leftpokeball"] = "leftpokeball";
  ItemsEnum2["Legendplate"] = "legendplate";
  ItemsEnum2["Lenscase"] = "lenscase";
  ItemsEnum2["Leppaberry"] = "leppaberry";
  ItemsEnum2["Letter"] = "letter";
  ItemsEnum2["Levelball"] = "levelball";
  ItemsEnum2["Libertypass"] = "libertypass";
  ItemsEnum2["Liechiberry"] = "liechiberry";
  ItemsEnum2["Lifeorb"] = "lifeorb";
  ItemsEnum2["Liftkey"] = "liftkey";
  ItemsEnum2["Lightball"] = "lightball";
  ItemsEnum2["Lightclay"] = "lightclay";
  ItemsEnum2["Lightstone"] = "lightstone";
  ItemsEnum2["Lockcapsule"] = "lockcapsule";
  ItemsEnum2["Lookerticket"] = "lookerticket";
  ItemsEnum2["Lootsack"] = "lootsack";
  ItemsEnum2["Lopunnite"] = "lopunnite";
  ItemsEnum2["Lostitem"] = "lostitem";
  ItemsEnum2["Loveball"] = "loveball";
  ItemsEnum2["Lovesweet"] = "lovesweet";
  ItemsEnum2["Lucarionite"] = "lucarionite";
  ItemsEnum2["Luckypunch"] = "luckypunch";
  ItemsEnum2["Lumberry"] = "lumberry";
  ItemsEnum2["Luminousmoss"] = "luminousmoss";
  ItemsEnum2["Lunaliumz"] = "lunaliumz";
  ItemsEnum2["Lunarwing"] = "lunarwing";
  ItemsEnum2["Lureball"] = "lureball";
  ItemsEnum2["Lustrousorb"] = "lustrousorb";
  ItemsEnum2["Luxuryball"] = "luxuryball";
  ItemsEnum2["Lycaniumz"] = "lycaniumz";
  ItemsEnum2["Machbike"] = "machbike";
  ItemsEnum2["Machinepart"] = "machinepart";
  ItemsEnum2["Machobrace"] = "machobrace";
  ItemsEnum2["Magmaemblem"] = "magmaemblem";
  ItemsEnum2["Magmarizer"] = "magmarizer";
  ItemsEnum2["Magmastone"] = "magmastone";
  ItemsEnum2["Magmasuit"] = "magmasuit";
  ItemsEnum2["Magnet"] = "magnet";
  ItemsEnum2["Magoberry"] = "magoberry";
  ItemsEnum2["Magostberry"] = "magostberry";
  ItemsEnum2["Mail"] = "mail";
  ItemsEnum2["Makeupbag"] = "makeupbag";
  ItemsEnum2["Manectite"] = "manectite";
  ItemsEnum2["Marangaberry"] = "marangaberry";
  ItemsEnum2["Marshadiumz"] = "marshadiumz";
  ItemsEnum2["Masterball"] = "masterball";
  ItemsEnum2["Mawilite"] = "mawilite";
  ItemsEnum2["Maxpotion"] = "maxpotion";
  ItemsEnum2["Meadowplate"] = "meadowplate";
  ItemsEnum2["Medalbox"] = "medalbox";
  ItemsEnum2["Medichamite"] = "medichamite";
  ItemsEnum2["Megabracelet"] = "megabracelet";
  ItemsEnum2["Megaring"] = "megaring";
  ItemsEnum2["Membercard"] = "membercard";
  ItemsEnum2["Mentalherb"] = "mentalherb";
  ItemsEnum2["Metagrossite"] = "metagrossite";
  ItemsEnum2["Metalcoat"] = "metalcoat";
  ItemsEnum2["Metalpowder"] = "metalpowder";
  ItemsEnum2["Meteorite"] = "meteorite";
  ItemsEnum2["Meteoriteshard"] = "meteoriteshard";
  ItemsEnum2["Metronome"] = "metronome";
  ItemsEnum2["Mewniumz"] = "mewniumz";
  ItemsEnum2["Mewtwonitex"] = "mewtwonitex";
  ItemsEnum2["Mewtwonitey"] = "mewtwonitey";
  ItemsEnum2["Micleberry"] = "micleberry";
  ItemsEnum2["Mimikiumz"] = "mimikiumz";
  ItemsEnum2["Mindplate"] = "mindplate";
  ItemsEnum2["Mintberry"] = "mintberry";
  ItemsEnum2["Miracleberry"] = "miracleberry";
  ItemsEnum2["Miracleseed"] = "miracleseed";
  ItemsEnum2["Mistyseed"] = "mistyseed";
  ItemsEnum2["Moonball"] = "moonball";
  ItemsEnum2["Moonflute"] = "moonflute";
  ItemsEnum2["Moonstone"] = "moonstone";
  ItemsEnum2["Muscleband"] = "muscleband";
  ItemsEnum2["Mysteryberry"] = "mysteryberry";
  ItemsEnum2["Mysteryegg"] = "mysteryegg";
  ItemsEnum2["Mysticticket"] = "mysticticket";
  ItemsEnum2["Mysticwater"] = "mysticwater";
  ItemsEnum2["Nanabberry"] = "nanabberry";
  ItemsEnum2["Nestball"] = "nestball";
  ItemsEnum2["Netball"] = "netball";
  ItemsEnum2["Nevermeltice"] = "nevermeltice";
  ItemsEnum2["Nlunarizer"] = "nlunarizer";
  ItemsEnum2["Nomelberry"] = "nomelberry";
  ItemsEnum2["Normalgem"] = "normalgem";
  ItemsEnum2["Normaliumz"] = "normaliumz";
  ItemsEnum2["Nsolarizer"] = "nsolarizer";
  ItemsEnum2["Oaksletter"] = "oaksletter";
  ItemsEnum2["Oaksparcel"] = "oaksparcel";
  ItemsEnum2["Occaberry"] = "occaberry";
  ItemsEnum2["Oddincense"] = "oddincense";
  ItemsEnum2["Oldamber"] = "oldamber";
  ItemsEnum2["Oldcharm"] = "oldcharm";
  ItemsEnum2["Oldletter"] = "oldletter";
  ItemsEnum2["Oldrod"] = "oldrod";
  ItemsEnum2["Oldseamap"] = "oldseamap";
  ItemsEnum2["Oranberry"] = "oranberry";
  ItemsEnum2["Orangepetal"] = "orangepetal";
  ItemsEnum2["Ovalcharm"] = "ovalcharm";
  ItemsEnum2["Ovalstone"] = "ovalstone";
  ItemsEnum2["Pairoftickets"] = "pairoftickets";
  ItemsEnum2["Palpad"] = "palpad";
  ItemsEnum2["Pamtreberry"] = "pamtreberry";
  ItemsEnum2["Parcel"] = "parcel";
  ItemsEnum2["Parkball"] = "parkball";
  ItemsEnum2["Pass"] = "pass";
  ItemsEnum2["Passhoberry"] = "passhoberry";
  ItemsEnum2["Payapaberry"] = "payapaberry";
  ItemsEnum2["Pechaberry"] = "pechaberry";
  ItemsEnum2["Permit"] = "permit";
  ItemsEnum2["Persimberry"] = "persimberry";
  ItemsEnum2["Petayaberry"] = "petayaberry";
  ItemsEnum2["Photoalbum"] = "photoalbum";
  ItemsEnum2["Pidgeotite"] = "pidgeotite";
  ItemsEnum2["Pikaniumz"] = "pikaniumz";
  ItemsEnum2["Pikashuniumz"] = "pikashuniumz";
  ItemsEnum2["Pinapberry"] = "pinapberry";
  ItemsEnum2["Pinkbow"] = "pinkbow";
  ItemsEnum2["Pinkpetal"] = "pinkpetal";
  ItemsEnum2["Pinsirite"] = "pinsirite";
  ItemsEnum2["Pixieplate"] = "pixieplate";
  ItemsEnum2["Plasmacard"] = "plasmacard";
  ItemsEnum2["Plumefossil"] = "plumefossil";
  ItemsEnum2["Poffincase"] = "poffincase";
  ItemsEnum2["Pointcard"] = "pointcard";
  ItemsEnum2["Poisonbarb"] = "poisonbarb";
  ItemsEnum2["Poisongem"] = "poisongem";
  ItemsEnum2["Poisoniumz"] = "poisoniumz";
  ItemsEnum2["Poisonmemory"] = "poisonmemory";
  ItemsEnum2["Pokeball"] = "pokeball";
  ItemsEnum2["Pokeblockcase"] = "pokeblockcase";
  ItemsEnum2["Pokeblockkit"] = "pokeblockkit";
  ItemsEnum2["Pokeflute"] = "pokeflute";
  ItemsEnum2["Pokemonboxlink"] = "pokemonboxlink";
  ItemsEnum2["Pokeradar"] = "pokeradar";
  ItemsEnum2["Polkadotbow"] = "polkadotbow";
  ItemsEnum2["Pomegberry"] = "pomegberry";
  ItemsEnum2["Potion"] = "potion";
  ItemsEnum2["Powderjar"] = "powderjar";
  ItemsEnum2["Poweranklet"] = "poweranklet";
  ItemsEnum2["Powerband"] = "powerband";
  ItemsEnum2["Powerbelt"] = "powerbelt";
  ItemsEnum2["Powerbracer"] = "powerbracer";
  ItemsEnum2["Powerherb"] = "powerherb";
  ItemsEnum2["Powerlens"] = "powerlens";
  ItemsEnum2["Powerplantpass"] = "powerplantpass";
  ItemsEnum2["Powerweight"] = "powerweight";
  ItemsEnum2["Premierball"] = "premierball";
  ItemsEnum2["Primariumz"] = "primariumz";
  ItemsEnum2["Prismscale"] = "prismscale";
  ItemsEnum2["Prisonbottle"] = "prisonbottle";
  ItemsEnum2["Professorsmask"] = "professorsmask";
  ItemsEnum2["Profsletter"] = "profsletter";
  ItemsEnum2["Propcase"] = "propcase";
  ItemsEnum2["Protectivepads"] = "protectivepads";
  ItemsEnum2["Protector"] = "protector";
  ItemsEnum2["Przcureberry"] = "przcureberry";
  ItemsEnum2["Psncureberry"] = "psncureberry";
  ItemsEnum2["Psychicgem"] = "psychicgem";
  ItemsEnum2["Psychicmemory"] = "psychicmemory";
  ItemsEnum2["Psychicseed"] = "psychicseed";
  ItemsEnum2["Psychiumz"] = "psychiumz";
  ItemsEnum2["Purplepetal"] = "purplepetal";
  ItemsEnum2["Qualotberry"] = "qualotberry";
  ItemsEnum2["Quickball"] = "quickball";
  ItemsEnum2["Quickclaw"] = "quickclaw";
  ItemsEnum2["Quickpowder"] = "quickpowder";
  ItemsEnum2["Rabutaberry"] = "rabutaberry";
  ItemsEnum2["Ragecandybar"] = "ragecandybar";
  ItemsEnum2["Rainbowflower"] = "rainbowflower";
  ItemsEnum2["Rainbowpass"] = "rainbowpass";
  ItemsEnum2["Rainbowwing"] = "rainbowwing";
  ItemsEnum2["Rarebone"] = "rarebone";
  ItemsEnum2["Rawstberry"] = "rawstberry";
  ItemsEnum2["Razorclaw"] = "razorclaw";
  ItemsEnum2["Razorfang"] = "razorfang";
  ItemsEnum2["Razzberry"] = "razzberry";
  ItemsEnum2["Reapercloth"] = "reapercloth";
  ItemsEnum2["Redcard"] = "redcard";
  ItemsEnum2["Redchain"] = "redchain";
  ItemsEnum2["Redorb"] = "redorb";
  ItemsEnum2["Redpetal"] = "redpetal";
  ItemsEnum2["Redscale"] = "redscale";
  ItemsEnum2["Repeatball"] = "repeatball";
  ItemsEnum2["Revealglass"] = "revealglass";
  ItemsEnum2["Ribbonsweet"] = "ribbonsweet";
  ItemsEnum2["Ridepager"] = "ridepager";
  ItemsEnum2["Rindoberry"] = "rindoberry";
  ItemsEnum2["Ringtarget"] = "ringtarget";
  ItemsEnum2["Rm1key"] = "rm1key";
  ItemsEnum2["Rm2key"] = "rm2key";
  ItemsEnum2["Rm4key"] = "rm4key";
  ItemsEnum2["Rm6key"] = "rm6key";
  ItemsEnum2["Rockgem"] = "rockgem";
  ItemsEnum2["Rockincense"] = "rockincense";
  ItemsEnum2["Rockiumz"] = "rockiumz";
  ItemsEnum2["Rockmemory"] = "rockmemory";
  ItemsEnum2["Rockyhelmet"] = "rockyhelmet";
  ItemsEnum2["Rollerskates"] = "rollerskates";
  ItemsEnum2["Roomservice"] = "roomservice";
  ItemsEnum2["Rootfossil"] = "rootfossil";
  ItemsEnum2["Roseincense"] = "roseincense";
  ItemsEnum2["Roseliberry"] = "roseliberry";
  ItemsEnum2["Rotombike"] = "rotombike";
  ItemsEnum2["Rotomcatalog"] = "rotomcatalog";
  ItemsEnum2["Rowapberry"] = "rowapberry";
  ItemsEnum2["Ruby"] = "ruby";
  ItemsEnum2["Rulebook"] = "rulebook";
  ItemsEnum2["Rustedshield"] = "rustedshield";
  ItemsEnum2["Rustedsword"] = "rustedsword";
  ItemsEnum2["Sablenite"] = "sablenite";
  ItemsEnum2["Sachet"] = "sachet";
  ItemsEnum2["Safariball"] = "safariball";
  ItemsEnum2["Safetygoggles"] = "safetygoggles";
  ItemsEnum2["Sailfossil"] = "sailfossil";
  ItemsEnum2["Salacberry"] = "salacberry";
  ItemsEnum2["Salamencite"] = "salamencite";
  ItemsEnum2["Sapphire"] = "sapphire";
  ItemsEnum2["Scanner"] = "scanner";
  ItemsEnum2["Sceptilite"] = "sceptilite";
  ItemsEnum2["Scizorite"] = "scizorite";
  ItemsEnum2["Scopelens"] = "scopelens";
  ItemsEnum2["Seaincense"] = "seaincense";
  ItemsEnum2["Sealbag"] = "sealbag";
  ItemsEnum2["Sealcase"] = "sealcase";
  ItemsEnum2["Secretkey"] = "secretkey";
  ItemsEnum2["Secretpotion"] = "secretpotion";
  ItemsEnum2["Sharpbeak"] = "sharpbeak";
  ItemsEnum2["Sharpedonite"] = "sharpedonite";
  ItemsEnum2["Shedshell"] = "shedshell";
  ItemsEnum2["Shellbell"] = "shellbell";
  ItemsEnum2["Shinycharm"] = "shinycharm";
  ItemsEnum2["Shinystone"] = "shinystone";
  ItemsEnum2["Shockdrive"] = "shockdrive";
  ItemsEnum2["Shucaberry"] = "shucaberry";
  ItemsEnum2["Silkscarf"] = "silkscarf";
  ItemsEnum2["Silphscope"] = "silphscope";
  ItemsEnum2["Silverpowder"] = "silverpowder";
  ItemsEnum2["Silverwing"] = "silverwing";
  ItemsEnum2["Sitrusberry"] = "sitrusberry";
  ItemsEnum2["Skullfossil"] = "skullfossil";
  ItemsEnum2["Skyplate"] = "skyplate";
  ItemsEnum2["Slowbronite"] = "slowbronite";
  ItemsEnum2["Slowpoketail"] = "slowpoketail";
  ItemsEnum2["Smoothrock"] = "smoothrock";
  ItemsEnum2["Snorliumz"] = "snorliumz";
  ItemsEnum2["Snowball"] = "snowball";
  ItemsEnum2["Softsand"] = "softsand";
  ItemsEnum2["Solganiumz"] = "solganiumz";
  ItemsEnum2["Soniasbook"] = "soniasbook";
  ItemsEnum2["Sootsack"] = "sootsack";
  ItemsEnum2["Souldew"] = "souldew";
  ItemsEnum2["Sparklingstone"] = "sparklingstone";
  ItemsEnum2["Spelltag"] = "spelltag";
  ItemsEnum2["Spelonberry"] = "spelonberry";
  ItemsEnum2["Splashplate"] = "splashplate";
  ItemsEnum2["Spookyplate"] = "spookyplate";
  ItemsEnum2["Sportball"] = "sportball";
  ItemsEnum2["Sprayduck"] = "sprayduck";
  ItemsEnum2["Sprinklotad"] = "sprinklotad";
  ItemsEnum2["Squirtbottle"] = "squirtbottle";
  ItemsEnum2["Ssticket"] = "ssticket";
  ItemsEnum2["Starfberry"] = "starfberry";
  ItemsEnum2["Starsweet"] = "starsweet";
  ItemsEnum2["Steelgem"] = "steelgem";
  ItemsEnum2["Steeliumz"] = "steeliumz";
  ItemsEnum2["Steelixite"] = "steelixite";
  ItemsEnum2["Steelmemory"] = "steelmemory";
  ItemsEnum2["Stick"] = "stick";
  ItemsEnum2["Stickybarb"] = "stickybarb";
  ItemsEnum2["Stoneplate"] = "stoneplate";
  ItemsEnum2["Storagekey"] = "storagekey";
  ItemsEnum2["Strawberrysweet"] = "strawberrysweet";
  ItemsEnum2["Suitekey"] = "suitekey";
  ItemsEnum2["Sunflute"] = "sunflute";
  ItemsEnum2["Sunstone"] = "sunstone";
  ItemsEnum2["Superpotion"] = "superpotion";
  ItemsEnum2["Superrod"] = "superrod";
  ItemsEnum2["Surgebadge"] = "surgebadge";
  ItemsEnum2["Swampertite"] = "swampertite";
  ItemsEnum2["Sweetapple"] = "sweetapple";
  ItemsEnum2["Tamatoberry"] = "tamatoberry";
  ItemsEnum2["Tangaberry"] = "tangaberry";
  ItemsEnum2["Tapuniumz"] = "tapuniumz";
  ItemsEnum2["Tartapple"] = "tartapple";
  ItemsEnum2["Tea"] = "tea";
  ItemsEnum2["Teachytv"] = "teachytv";
  ItemsEnum2["Terrainextender"] = "terrainextender";
  ItemsEnum2["Thickclub"] = "thickclub";
  ItemsEnum2["Throatspray"] = "throatspray";
  ItemsEnum2["Thunderstone"] = "thunderstone";
  ItemsEnum2["Tidalbell"] = "tidalbell";
  ItemsEnum2["Timerball"] = "timerball";
  ItemsEnum2["Tmcase"] = "tmcase";
  ItemsEnum2["Tmvpass"] = "tmvpass";
  ItemsEnum2["Townmap"] = "townmap";
  ItemsEnum2["Toxicorb"] = "toxicorb";
  ItemsEnum2["Toxicplate"] = "toxicplate";
  ItemsEnum2["Traveltrunk"] = "traveltrunk";
  ItemsEnum2["Tripass"] = "tripass";
  ItemsEnum2["Twistedspoon"] = "twistedspoon";
  ItemsEnum2["Tyranitarite"] = "tyranitarite";
  ItemsEnum2["Ultraball"] = "ultraball";
  ItemsEnum2["Ultranecroziumz"] = "ultranecroziumz";
  ItemsEnum2["Unownreport"] = "unownreport";
  ItemsEnum2["Upgrade"] = "upgrade";
  ItemsEnum2["Utilityumbrella"] = "utilityumbrella";
  ItemsEnum2["Venusaurite"] = "venusaurite";
  ItemsEnum2["Vilevial"] = "vilevial";
  ItemsEnum2["Vsrecorder"] = "vsrecorder";
  ItemsEnum2["Vsseeker"] = "vsseeker";
  ItemsEnum2["Wacanberry"] = "wacanberry";
  ItemsEnum2["Wailmerpail"] = "wailmerpail";
  ItemsEnum2["Watergem"] = "watergem";
  ItemsEnum2["Wateriumz"] = "wateriumz";
  ItemsEnum2["Watermemory"] = "watermemory";
  ItemsEnum2["Waterstone"] = "waterstone";
  ItemsEnum2["Watmelberry"] = "watmelberry";
  ItemsEnum2["Waveincense"] = "waveincense";
  ItemsEnum2["Weaknesspolicy"] = "weaknesspolicy";
  ItemsEnum2["Wepearberry"] = "wepearberry";
  ItemsEnum2["Whippeddream"] = "whippeddream";
  ItemsEnum2["Whiteherb"] = "whiteherb";
  ItemsEnum2["Widelens"] = "widelens";
  ItemsEnum2["Wikiberry"] = "wikiberry";
  ItemsEnum2["Wiseglasses"] = "wiseglasses";
  ItemsEnum2["Wishingstar"] = "wishingstar";
  ItemsEnum2["Workskey"] = "workskey";
  ItemsEnum2["Xtransceiver"] = "xtransceiver";
  ItemsEnum2["Yacheberry"] = "yacheberry";
  ItemsEnum2["Yellowpetal"] = "yellowpetal";
  ItemsEnum2["Zapplate"] = "zapplate";
  ItemsEnum2["Zoomlens"] = "zoomlens";
  ItemsEnum2["Zpowering"] = "zpowering";
  ItemsEnum2["Zring"] = "zring";
  ItemsEnum2["Zygardecube"] = "zygardecube";
  return ItemsEnum2;
})(ItemsEnum || {});
var MovesEnum = /* @__PURE__ */ ((MovesEnum2) => {
  MovesEnum2["Absorb"] = "absorb";
  MovesEnum2["Accelerock"] = "accelerock";
  MovesEnum2["Acid"] = "acid";
  MovesEnum2["Acidarmor"] = "acidarmor";
  MovesEnum2["Aciddownpour"] = "aciddownpour";
  MovesEnum2["Acidspray"] = "acidspray";
  MovesEnum2["Acrobatics"] = "acrobatics";
  MovesEnum2["Acupressure"] = "acupressure";
  MovesEnum2["Aerialace"] = "aerialace";
  MovesEnum2["Aeroblast"] = "aeroblast";
  MovesEnum2["Afteryou"] = "afteryou";
  MovesEnum2["Agility"] = "agility";
  MovesEnum2["Aircutter"] = "aircutter";
  MovesEnum2["Airslash"] = "airslash";
  MovesEnum2["Alloutpummeling"] = "alloutpummeling";
  MovesEnum2["Allyswitch"] = "allyswitch";
  MovesEnum2["Amnesia"] = "amnesia";
  MovesEnum2["Anchorshot"] = "anchorshot";
  MovesEnum2["Ancientpower"] = "ancientpower";
  MovesEnum2["Appleacid"] = "appleacid";
  MovesEnum2["Aquajet"] = "aquajet";
  MovesEnum2["Aquaring"] = "aquaring";
  MovesEnum2["Aquatail"] = "aquatail";
  MovesEnum2["Armthrust"] = "armthrust";
  MovesEnum2["Aromatherapy"] = "aromatherapy";
  MovesEnum2["Aromaticmist"] = "aromaticmist";
  MovesEnum2["Assist"] = "assist";
  MovesEnum2["Assurance"] = "assurance";
  MovesEnum2["Astonish"] = "astonish";
  MovesEnum2["Astralbarrage"] = "astralbarrage";
  MovesEnum2["Attackorder"] = "attackorder";
  MovesEnum2["Attract"] = "attract";
  MovesEnum2["Aurasphere"] = "aurasphere";
  MovesEnum2["Aurawheel"] = "aurawheel";
  MovesEnum2["Aurorabeam"] = "aurorabeam";
  MovesEnum2["Auroraveil"] = "auroraveil";
  MovesEnum2["Autotomize"] = "autotomize";
  MovesEnum2["Avalanche"] = "avalanche";
  MovesEnum2["Babydolleyes"] = "babydolleyes";
  MovesEnum2["Baddybad"] = "baddybad";
  MovesEnum2["Banefulbunker"] = "banefulbunker";
  MovesEnum2["Barbbarrage"] = "barbbarrage";
  MovesEnum2["Barrage"] = "barrage";
  MovesEnum2["Barrier"] = "barrier";
  MovesEnum2["Batonpass"] = "batonpass";
  MovesEnum2["Beakblast"] = "beakblast";
  MovesEnum2["Beatup"] = "beatup";
  MovesEnum2["Behemothbash"] = "behemothbash";
  MovesEnum2["Behemothblade"] = "behemothblade";
  MovesEnum2["Belch"] = "belch";
  MovesEnum2["Bellydrum"] = "bellydrum";
  MovesEnum2["Bestow"] = "bestow";
  MovesEnum2["Bide"] = "bide";
  MovesEnum2["Bind"] = "bind";
  MovesEnum2["Bite"] = "bite";
  MovesEnum2["Bittermalice"] = "bittermalice";
  MovesEnum2["Blackholeeclipse"] = "blackholeeclipse";
  MovesEnum2["Blastburn"] = "blastburn";
  MovesEnum2["Blazekick"] = "blazekick";
  MovesEnum2["Bleakwindstorm"] = "bleakwindstorm";
  MovesEnum2["Blizzard"] = "blizzard";
  MovesEnum2["Block"] = "block";
  MovesEnum2["Bloomdoom"] = "bloomdoom";
  MovesEnum2["Blueflare"] = "blueflare";
  MovesEnum2["Bodypress"] = "bodypress";
  MovesEnum2["Bodyslam"] = "bodyslam";
  MovesEnum2["Boltbeak"] = "boltbeak";
  MovesEnum2["Boltstrike"] = "boltstrike";
  MovesEnum2["Boneclub"] = "boneclub";
  MovesEnum2["Bonemerang"] = "bonemerang";
  MovesEnum2["Bonerush"] = "bonerush";
  MovesEnum2["Boomburst"] = "boomburst";
  MovesEnum2["Bounce"] = "bounce";
  MovesEnum2["Bouncybubble"] = "bouncybubble";
  MovesEnum2["Branchpoke"] = "branchpoke";
  MovesEnum2["Bravebird"] = "bravebird";
  MovesEnum2["Breakingswipe"] = "breakingswipe";
  MovesEnum2["Breakneckblitz"] = "breakneckblitz";
  MovesEnum2["Brickbreak"] = "brickbreak";
  MovesEnum2["Brine"] = "brine";
  MovesEnum2["Brutalswing"] = "brutalswing";
  MovesEnum2["Bubble"] = "bubble";
  MovesEnum2["Bubblebeam"] = "bubblebeam";
  MovesEnum2["Bugbite"] = "bugbite";
  MovesEnum2["Bugbuzz"] = "bugbuzz";
  MovesEnum2["Bulkup"] = "bulkup";
  MovesEnum2["Bulldoze"] = "bulldoze";
  MovesEnum2["Bulletpunch"] = "bulletpunch";
  MovesEnum2["Bulletseed"] = "bulletseed";
  MovesEnum2["Burningjealousy"] = "burningjealousy";
  MovesEnum2["Burnup"] = "burnup";
  MovesEnum2["Buzzybuzz"] = "buzzybuzz";
  MovesEnum2["Calmmind"] = "calmmind";
  MovesEnum2["Camouflage"] = "camouflage";
  MovesEnum2["Captivate"] = "captivate";
  MovesEnum2["Catastropika"] = "catastropika";
  MovesEnum2["Ceaselessedge"] = "ceaselessedge";
  MovesEnum2["Celebrate"] = "celebrate";
  MovesEnum2["Charge"] = "charge";
  MovesEnum2["Chargebeam"] = "chargebeam";
  MovesEnum2["Charm"] = "charm";
  MovesEnum2["Chatter"] = "chatter";
  MovesEnum2["Chipaway"] = "chipaway";
  MovesEnum2["Chloroblast"] = "chloroblast";
  MovesEnum2["Circlethrow"] = "circlethrow";
  MovesEnum2["Clamp"] = "clamp";
  MovesEnum2["Clangingscales"] = "clangingscales";
  MovesEnum2["Clangoroussoul"] = "clangoroussoul";
  MovesEnum2["Clangoroussoulblaze"] = "clangoroussoulblaze";
  MovesEnum2["Clearsmog"] = "clearsmog";
  MovesEnum2["Closecombat"] = "closecombat";
  MovesEnum2["Coaching"] = "coaching";
  MovesEnum2["Coil"] = "coil";
  MovesEnum2["Cometpunch"] = "cometpunch";
  MovesEnum2["Confide"] = "confide";
  MovesEnum2["Confuseray"] = "confuseray";
  MovesEnum2["Confusion"] = "confusion";
  MovesEnum2["Constrict"] = "constrict";
  MovesEnum2["Continentalcrush"] = "continentalcrush";
  MovesEnum2["Conversion"] = "conversion";
  MovesEnum2["Conversion2"] = "conversion2";
  MovesEnum2["Copycat"] = "copycat";
  MovesEnum2["Coreenforcer"] = "coreenforcer";
  MovesEnum2["Corkscrewcrash"] = "corkscrewcrash";
  MovesEnum2["Corrosivegas"] = "corrosivegas";
  MovesEnum2["Cosmicpower"] = "cosmicpower";
  MovesEnum2["Cottonguard"] = "cottonguard";
  MovesEnum2["Cottonspore"] = "cottonspore";
  MovesEnum2["Counter"] = "counter";
  MovesEnum2["Courtchange"] = "courtchange";
  MovesEnum2["Covet"] = "covet";
  MovesEnum2["Crabhammer"] = "crabhammer";
  MovesEnum2["Craftyshield"] = "craftyshield";
  MovesEnum2["Crosschop"] = "crosschop";
  MovesEnum2["Crosspoison"] = "crosspoison";
  MovesEnum2["Crunch"] = "crunch";
  MovesEnum2["Crushclaw"] = "crushclaw";
  MovesEnum2["Crushgrip"] = "crushgrip";
  MovesEnum2["Curse"] = "curse";
  MovesEnum2["Cut"] = "cut";
  MovesEnum2["Darkestlariat"] = "darkestlariat";
  MovesEnum2["Darkpulse"] = "darkpulse";
  MovesEnum2["Darkvoid"] = "darkvoid";
  MovesEnum2["Dazzlinggleam"] = "dazzlinggleam";
  MovesEnum2["Decorate"] = "decorate";
  MovesEnum2["Defendorder"] = "defendorder";
  MovesEnum2["Defensecurl"] = "defensecurl";
  MovesEnum2["Defog"] = "defog";
  MovesEnum2["Destinybond"] = "destinybond";
  MovesEnum2["Detect"] = "detect";
  MovesEnum2["Devastatingdrake"] = "devastatingdrake";
  MovesEnum2["Diamondstorm"] = "diamondstorm";
  MovesEnum2["Dig"] = "dig";
  MovesEnum2["Direclaw"] = "direclaw";
  MovesEnum2["Disable"] = "disable";
  MovesEnum2["Disarmingvoice"] = "disarmingvoice";
  MovesEnum2["Discharge"] = "discharge";
  MovesEnum2["Dive"] = "dive";
  MovesEnum2["Dizzypunch"] = "dizzypunch";
  MovesEnum2["Doomdesire"] = "doomdesire";
  MovesEnum2["Doubleedge"] = "doubleedge";
  MovesEnum2["Doublehit"] = "doublehit";
  MovesEnum2["Doubleironbash"] = "doubleironbash";
  MovesEnum2["Doublekick"] = "doublekick";
  MovesEnum2["Doubleslap"] = "doubleslap";
  MovesEnum2["Doubleteam"] = "doubleteam";
  MovesEnum2["Dracometeor"] = "dracometeor";
  MovesEnum2["Dragonascent"] = "dragonascent";
  MovesEnum2["Dragonbreath"] = "dragonbreath";
  MovesEnum2["Dragonclaw"] = "dragonclaw";
  MovesEnum2["Dragondance"] = "dragondance";
  MovesEnum2["Dragondarts"] = "dragondarts";
  MovesEnum2["Dragonenergy"] = "dragonenergy";
  MovesEnum2["Dragonhammer"] = "dragonhammer";
  MovesEnum2["Dragonpulse"] = "dragonpulse";
  MovesEnum2["Dragonrage"] = "dragonrage";
  MovesEnum2["Dragonrush"] = "dragonrush";
  MovesEnum2["Dragontail"] = "dragontail";
  MovesEnum2["Drainingkiss"] = "drainingkiss";
  MovesEnum2["Drainpunch"] = "drainpunch";
  MovesEnum2["Dreameater"] = "dreameater";
  MovesEnum2["Drillpeck"] = "drillpeck";
  MovesEnum2["Drillrun"] = "drillrun";
  MovesEnum2["Drumbeating"] = "drumbeating";
  MovesEnum2["Dualchop"] = "dualchop";
  MovesEnum2["Dualwingbeat"] = "dualwingbeat";
  MovesEnum2["Dynamaxcannon"] = "dynamaxcannon";
  MovesEnum2["Dynamicpunch"] = "dynamicpunch";
  MovesEnum2["Earthpower"] = "earthpower";
  MovesEnum2["Earthquake"] = "earthquake";
  MovesEnum2["Echoedvoice"] = "echoedvoice";
  MovesEnum2["Eerieimpulse"] = "eerieimpulse";
  MovesEnum2["Eeriespell"] = "eeriespell";
  MovesEnum2["Eggbomb"] = "eggbomb";
  MovesEnum2["Electricterrain"] = "electricterrain";
  MovesEnum2["Electrify"] = "electrify";
  MovesEnum2["Electroball"] = "electroball";
  MovesEnum2["Electroweb"] = "electroweb";
  MovesEnum2["Embargo"] = "embargo";
  MovesEnum2["Ember"] = "ember";
  MovesEnum2["Encore"] = "encore";
  MovesEnum2["Endeavor"] = "endeavor";
  MovesEnum2["Endure"] = "endure";
  MovesEnum2["Energyball"] = "energyball";
  MovesEnum2["Entrainment"] = "entrainment";
  MovesEnum2["Eruption"] = "eruption";
  MovesEnum2["Esperwing"] = "esperwing";
  MovesEnum2["Eternabeam"] = "eternabeam";
  MovesEnum2["Expandingforce"] = "expandingforce";
  MovesEnum2["Explosion"] = "explosion";
  MovesEnum2["Extrasensory"] = "extrasensory";
  MovesEnum2["Extremeevoboost"] = "extremeevoboost";
  MovesEnum2["Extremespeed"] = "extremespeed";
  MovesEnum2["Facade"] = "facade";
  MovesEnum2["Fairylock"] = "fairylock";
  MovesEnum2["Fairywind"] = "fairywind";
  MovesEnum2["Fakeout"] = "fakeout";
  MovesEnum2["Faketears"] = "faketears";
  MovesEnum2["Falsesurrender"] = "falsesurrender";
  MovesEnum2["Falseswipe"] = "falseswipe";
  MovesEnum2["Featherdance"] = "featherdance";
  MovesEnum2["Feint"] = "feint";
  MovesEnum2["Feintattack"] = "feintattack";
  MovesEnum2["Fellstinger"] = "fellstinger";
  MovesEnum2["Fierydance"] = "fierydance";
  MovesEnum2["Fierywrath"] = "fierywrath";
  MovesEnum2["Finalgambit"] = "finalgambit";
  MovesEnum2["Fireblast"] = "fireblast";
  MovesEnum2["Firefang"] = "firefang";
  MovesEnum2["Firelash"] = "firelash";
  MovesEnum2["Firepledge"] = "firepledge";
  MovesEnum2["Firepunch"] = "firepunch";
  MovesEnum2["Firespin"] = "firespin";
  MovesEnum2["Firstimpression"] = "firstimpression";
  MovesEnum2["Fishiousrend"] = "fishiousrend";
  MovesEnum2["Fissure"] = "fissure";
  MovesEnum2["Flail"] = "flail";
  MovesEnum2["Flameburst"] = "flameburst";
  MovesEnum2["Flamecharge"] = "flamecharge";
  MovesEnum2["Flamethrower"] = "flamethrower";
  MovesEnum2["Flamewheel"] = "flamewheel";
  MovesEnum2["Flareblitz"] = "flareblitz";
  MovesEnum2["Flash"] = "flash";
  MovesEnum2["Flashcannon"] = "flashcannon";
  MovesEnum2["Flatter"] = "flatter";
  MovesEnum2["Fleurcannon"] = "fleurcannon";
  MovesEnum2["Fling"] = "fling";
  MovesEnum2["Flipturn"] = "flipturn";
  MovesEnum2["Floatyfall"] = "floatyfall";
  MovesEnum2["Floralhealing"] = "floralhealing";
  MovesEnum2["Flowershield"] = "flowershield";
  MovesEnum2["Fly"] = "fly";
  MovesEnum2["Flyingpress"] = "flyingpress";
  MovesEnum2["Focusblast"] = "focusblast";
  MovesEnum2["Focusenergy"] = "focusenergy";
  MovesEnum2["Focuspunch"] = "focuspunch";
  MovesEnum2["Followme"] = "followme";
  MovesEnum2["Forcepalm"] = "forcepalm";
  MovesEnum2["Foresight"] = "foresight";
  MovesEnum2["Forestscurse"] = "forestscurse";
  MovesEnum2["Foulplay"] = "foulplay";
  MovesEnum2["Freezedry"] = "freezedry";
  MovesEnum2["Freezeshock"] = "freezeshock";
  MovesEnum2["Freezingglare"] = "freezingglare";
  MovesEnum2["Freezyfrost"] = "freezyfrost";
  MovesEnum2["Frenzyplant"] = "frenzyplant";
  MovesEnum2["Frostbreath"] = "frostbreath";
  MovesEnum2["Frustration"] = "frustration";
  MovesEnum2["Furyattack"] = "furyattack";
  MovesEnum2["Furycutter"] = "furycutter";
  MovesEnum2["Furyswipes"] = "furyswipes";
  MovesEnum2["Fusionbolt"] = "fusionbolt";
  MovesEnum2["Fusionflare"] = "fusionflare";
  MovesEnum2["Futuresight"] = "futuresight";
  MovesEnum2["Gastroacid"] = "gastroacid";
  MovesEnum2["Geargrind"] = "geargrind";
  MovesEnum2["Gearup"] = "gearup";
  MovesEnum2["Genesissupernova"] = "genesissupernova";
  MovesEnum2["Geomancy"] = "geomancy";
  MovesEnum2["Gigadrain"] = "gigadrain";
  MovesEnum2["Gigaimpact"] = "gigaimpact";
  MovesEnum2["Gigavolthavoc"] = "gigavolthavoc";
  MovesEnum2["Glaciallance"] = "glaciallance";
  MovesEnum2["Glaciate"] = "glaciate";
  MovesEnum2["Glare"] = "glare";
  MovesEnum2["Glitzyglow"] = "glitzyglow";
  MovesEnum2["Gmaxbefuddle"] = "gmaxbefuddle";
  MovesEnum2["Gmaxcannonade"] = "gmaxcannonade";
  MovesEnum2["Gmaxcentiferno"] = "gmaxcentiferno";
  MovesEnum2["Gmaxchistrike"] = "gmaxchistrike";
  MovesEnum2["Gmaxcuddle"] = "gmaxcuddle";
  MovesEnum2["Gmaxdepletion"] = "gmaxdepletion";
  MovesEnum2["Gmaxdrumsolo"] = "gmaxdrumsolo";
  MovesEnum2["Gmaxfinale"] = "gmaxfinale";
  MovesEnum2["Gmaxfireball"] = "gmaxfireball";
  MovesEnum2["Gmaxfoamburst"] = "gmaxfoamburst";
  MovesEnum2["Gmaxgoldrush"] = "gmaxgoldrush";
  MovesEnum2["Gmaxgravitas"] = "gmaxgravitas";
  MovesEnum2["Gmaxhydrosnipe"] = "gmaxhydrosnipe";
  MovesEnum2["Gmaxmalodor"] = "gmaxmalodor";
  MovesEnum2["Gmaxmeltdown"] = "gmaxmeltdown";
  MovesEnum2["Gmaxoneblow"] = "gmaxoneblow";
  MovesEnum2["Gmaxrapidflow"] = "gmaxrapidflow";
  MovesEnum2["Gmaxreplenish"] = "gmaxreplenish";
  MovesEnum2["Gmaxresonance"] = "gmaxresonance";
  MovesEnum2["Gmaxsandblast"] = "gmaxsandblast";
  MovesEnum2["Gmaxsmite"] = "gmaxsmite";
  MovesEnum2["Gmaxsnooze"] = "gmaxsnooze";
  MovesEnum2["Gmaxsteelsurge"] = "gmaxsteelsurge";
  MovesEnum2["Gmaxstonesurge"] = "gmaxstonesurge";
  MovesEnum2["Gmaxstunshock"] = "gmaxstunshock";
  MovesEnum2["Gmaxsweetness"] = "gmaxsweetness";
  MovesEnum2["Gmaxtartness"] = "gmaxtartness";
  MovesEnum2["Gmaxterror"] = "gmaxterror";
  MovesEnum2["Gmaxvinelash"] = "gmaxvinelash";
  MovesEnum2["Gmaxvolcalith"] = "gmaxvolcalith";
  MovesEnum2["Gmaxvoltcrash"] = "gmaxvoltcrash";
  MovesEnum2["Gmaxwildfire"] = "gmaxwildfire";
  MovesEnum2["Gmaxwindrage"] = "gmaxwindrage";
  MovesEnum2["Grassknot"] = "grassknot";
  MovesEnum2["Grasspledge"] = "grasspledge";
  MovesEnum2["Grasswhistle"] = "grasswhistle";
  MovesEnum2["Grassyglide"] = "grassyglide";
  MovesEnum2["Grassyterrain"] = "grassyterrain";
  MovesEnum2["Gravapple"] = "gravapple";
  MovesEnum2["Gravity"] = "gravity";
  MovesEnum2["Growl"] = "growl";
  MovesEnum2["Growth"] = "growth";
  MovesEnum2["Grudge"] = "grudge";
  MovesEnum2["Guardianofalola"] = "guardianofalola";
  MovesEnum2["Guardsplit"] = "guardsplit";
  MovesEnum2["Guardswap"] = "guardswap";
  MovesEnum2["Guillotine"] = "guillotine";
  MovesEnum2["Gunkshot"] = "gunkshot";
  MovesEnum2["Gust"] = "gust";
  MovesEnum2["Gyroball"] = "gyroball";
  MovesEnum2["Hail"] = "hail";
  MovesEnum2["Hammerarm"] = "hammerarm";
  MovesEnum2["Happyhour"] = "happyhour";
  MovesEnum2["Harden"] = "harden";
  MovesEnum2["Haze"] = "haze";
  MovesEnum2["Headbutt"] = "headbutt";
  MovesEnum2["Headcharge"] = "headcharge";
  MovesEnum2["Headlongrush"] = "headlongrush";
  MovesEnum2["Headsmash"] = "headsmash";
  MovesEnum2["Healbell"] = "healbell";
  MovesEnum2["Healblock"] = "healblock";
  MovesEnum2["Healingwish"] = "healingwish";
  MovesEnum2["Healorder"] = "healorder";
  MovesEnum2["Healpulse"] = "healpulse";
  MovesEnum2["Heartstamp"] = "heartstamp";
  MovesEnum2["Heartswap"] = "heartswap";
  MovesEnum2["Heatcrash"] = "heatcrash";
  MovesEnum2["Heatwave"] = "heatwave";
  MovesEnum2["Heavyslam"] = "heavyslam";
  MovesEnum2["Helpinghand"] = "helpinghand";
  MovesEnum2["Hex"] = "hex";
  MovesEnum2["Hiddenpower"] = "hiddenpower";
  MovesEnum2["Hiddenpowerbug"] = "hiddenpowerbug";
  MovesEnum2["Hiddenpowerdark"] = "hiddenpowerdark";
  MovesEnum2["Hiddenpowerdragon"] = "hiddenpowerdragon";
  MovesEnum2["Hiddenpowerelectric"] = "hiddenpowerelectric";
  MovesEnum2["Hiddenpowerfighting"] = "hiddenpowerfighting";
  MovesEnum2["Hiddenpowerfire"] = "hiddenpowerfire";
  MovesEnum2["Hiddenpowerflying"] = "hiddenpowerflying";
  MovesEnum2["Hiddenpowerghost"] = "hiddenpowerghost";
  MovesEnum2["Hiddenpowergrass"] = "hiddenpowergrass";
  MovesEnum2["Hiddenpowerground"] = "hiddenpowerground";
  MovesEnum2["Hiddenpowerice"] = "hiddenpowerice";
  MovesEnum2["Hiddenpowerpoison"] = "hiddenpowerpoison";
  MovesEnum2["Hiddenpowerpsychic"] = "hiddenpowerpsychic";
  MovesEnum2["Hiddenpowerrock"] = "hiddenpowerrock";
  MovesEnum2["Hiddenpowersteel"] = "hiddenpowersteel";
  MovesEnum2["Hiddenpowerwater"] = "hiddenpowerwater";
  MovesEnum2["Highhorsepower"] = "highhorsepower";
  MovesEnum2["Highjumpkick"] = "highjumpkick";
  MovesEnum2["Holdback"] = "holdback";
  MovesEnum2["Holdhands"] = "holdhands";
  MovesEnum2["Honeclaws"] = "honeclaws";
  MovesEnum2["Hornattack"] = "hornattack";
  MovesEnum2["Horndrill"] = "horndrill";
  MovesEnum2["Hornleech"] = "hornleech";
  MovesEnum2["Howl"] = "howl";
  MovesEnum2["Hurricane"] = "hurricane";
  MovesEnum2["Hydrocannon"] = "hydrocannon";
  MovesEnum2["Hydropump"] = "hydropump";
  MovesEnum2["Hydrovortex"] = "hydrovortex";
  MovesEnum2["Hyperbeam"] = "hyperbeam";
  MovesEnum2["Hyperfang"] = "hyperfang";
  MovesEnum2["Hyperspacefury"] = "hyperspacefury";
  MovesEnum2["Hyperspacehole"] = "hyperspacehole";
  MovesEnum2["Hypervoice"] = "hypervoice";
  MovesEnum2["Hypnosis"] = "hypnosis";
  MovesEnum2["Iceball"] = "iceball";
  MovesEnum2["Icebeam"] = "icebeam";
  MovesEnum2["Iceburn"] = "iceburn";
  MovesEnum2["Icefang"] = "icefang";
  MovesEnum2["Icehammer"] = "icehammer";
  MovesEnum2["Icepunch"] = "icepunch";
  MovesEnum2["Iceshard"] = "iceshard";
  MovesEnum2["Iciclecrash"] = "iciclecrash";
  MovesEnum2["Iciclespear"] = "iciclespear";
  MovesEnum2["Icywind"] = "icywind";
  MovesEnum2["Imprison"] = "imprison";
  MovesEnum2["Incinerate"] = "incinerate";
  MovesEnum2["Infernalparade"] = "infernalparade";
  MovesEnum2["Inferno"] = "inferno";
  MovesEnum2["Infernooverdrive"] = "infernooverdrive";
  MovesEnum2["Infestation"] = "infestation";
  MovesEnum2["Ingrain"] = "ingrain";
  MovesEnum2["Instruct"] = "instruct";
  MovesEnum2["Iondeluge"] = "iondeluge";
  MovesEnum2["Irondefense"] = "irondefense";
  MovesEnum2["Ironhead"] = "ironhead";
  MovesEnum2["Irontail"] = "irontail";
  MovesEnum2["Jawlock"] = "jawlock";
  MovesEnum2["Judgment"] = "judgment";
  MovesEnum2["Jumpkick"] = "jumpkick";
  MovesEnum2["Junglehealing"] = "junglehealing";
  MovesEnum2["Karatechop"] = "karatechop";
  MovesEnum2["Kinesis"] = "kinesis";
  MovesEnum2["Kingsshield"] = "kingsshield";
  MovesEnum2["Knockoff"] = "knockoff";
  MovesEnum2["Landswrath"] = "landswrath";
  MovesEnum2["Laserfocus"] = "laserfocus";
  MovesEnum2["Lashout"] = "lashout";
  MovesEnum2["Lastresort"] = "lastresort";
  MovesEnum2["Lavaplume"] = "lavaplume";
  MovesEnum2["Leafage"] = "leafage";
  MovesEnum2["Leafblade"] = "leafblade";
  MovesEnum2["Leafstorm"] = "leafstorm";
  MovesEnum2["Leaftornado"] = "leaftornado";
  MovesEnum2["Leechlife"] = "leechlife";
  MovesEnum2["Leechseed"] = "leechseed";
  MovesEnum2["Leer"] = "leer";
  MovesEnum2["Letssnuggleforever"] = "letssnuggleforever";
  MovesEnum2["Lick"] = "lick";
  MovesEnum2["Lifedew"] = "lifedew";
  MovesEnum2["Lightofruin"] = "lightofruin";
  MovesEnum2["Lightscreen"] = "lightscreen";
  MovesEnum2["Lightthatburnsthesky"] = "lightthatburnsthesky";
  MovesEnum2["Liquidation"] = "liquidation";
  MovesEnum2["Lockon"] = "lockon";
  MovesEnum2["Lovelykiss"] = "lovelykiss";
  MovesEnum2["Lowkick"] = "lowkick";
  MovesEnum2["Lowsweep"] = "lowsweep";
  MovesEnum2["Luckychant"] = "luckychant";
  MovesEnum2["Lunarblessing"] = "lunarblessing";
  MovesEnum2["Lunardance"] = "lunardance";
  MovesEnum2["Lunge"] = "lunge";
  MovesEnum2["Lusterpurge"] = "lusterpurge";
  MovesEnum2["Machpunch"] = "machpunch";
  MovesEnum2["Magicalleaf"] = "magicalleaf";
  MovesEnum2["Magiccoat"] = "magiccoat";
  MovesEnum2["Magicpowder"] = "magicpowder";
  MovesEnum2["Magicroom"] = "magicroom";
  MovesEnum2["Magikarpsrevenge"] = "magikarpsrevenge";
  MovesEnum2["Magmastorm"] = "magmastorm";
  MovesEnum2["Magnetbomb"] = "magnetbomb";
  MovesEnum2["Magneticflux"] = "magneticflux";
  MovesEnum2["Magnetrise"] = "magnetrise";
  MovesEnum2["Magnitude"] = "magnitude";
  MovesEnum2["Maliciousmoonsault"] = "maliciousmoonsault";
  MovesEnum2["Matblock"] = "matblock";
  MovesEnum2["Maxairstream"] = "maxairstream";
  MovesEnum2["Maxdarkness"] = "maxdarkness";
  MovesEnum2["Maxflare"] = "maxflare";
  MovesEnum2["Maxflutterby"] = "maxflutterby";
  MovesEnum2["Maxgeyser"] = "maxgeyser";
  MovesEnum2["Maxguard"] = "maxguard";
  MovesEnum2["Maxhailstorm"] = "maxhailstorm";
  MovesEnum2["Maxknuckle"] = "maxknuckle";
  MovesEnum2["Maxlightning"] = "maxlightning";
  MovesEnum2["Maxmindstorm"] = "maxmindstorm";
  MovesEnum2["Maxooze"] = "maxooze";
  MovesEnum2["Maxovergrowth"] = "maxovergrowth";
  MovesEnum2["Maxphantasm"] = "maxphantasm";
  MovesEnum2["Maxquake"] = "maxquake";
  MovesEnum2["Maxrockfall"] = "maxrockfall";
  MovesEnum2["Maxstarfall"] = "maxstarfall";
  MovesEnum2["Maxsteelspike"] = "maxsteelspike";
  MovesEnum2["Maxstrike"] = "maxstrike";
  MovesEnum2["Maxwyrmwind"] = "maxwyrmwind";
  MovesEnum2["Meanlook"] = "meanlook";
  MovesEnum2["Meditate"] = "meditate";
  MovesEnum2["Mefirst"] = "mefirst";
  MovesEnum2["Megadrain"] = "megadrain";
  MovesEnum2["Megahorn"] = "megahorn";
  MovesEnum2["Megakick"] = "megakick";
  MovesEnum2["Megapunch"] = "megapunch";
  MovesEnum2["Memento"] = "memento";
  MovesEnum2["Menacingmoonrazemaelstrom"] = "menacingmoonrazemaelstrom";
  MovesEnum2["Metalburst"] = "metalburst";
  MovesEnum2["Metalclaw"] = "metalclaw";
  MovesEnum2["Metalsound"] = "metalsound";
  MovesEnum2["Meteorassault"] = "meteorassault";
  MovesEnum2["Meteorbeam"] = "meteorbeam";
  MovesEnum2["Meteormash"] = "meteormash";
  MovesEnum2["Metronome"] = "metronome";
  MovesEnum2["Milkdrink"] = "milkdrink";
  MovesEnum2["Mimic"] = "mimic";
  MovesEnum2["Mindblown"] = "mindblown";
  MovesEnum2["Mindreader"] = "mindreader";
  MovesEnum2["Minimize"] = "minimize";
  MovesEnum2["Miracleeye"] = "miracleeye";
  MovesEnum2["Mirrorcoat"] = "mirrorcoat";
  MovesEnum2["Mirrormove"] = "mirrormove";
  MovesEnum2["Mirrorshot"] = "mirrorshot";
  MovesEnum2["Mist"] = "mist";
  MovesEnum2["Mistball"] = "mistball";
  MovesEnum2["Mistyexplosion"] = "mistyexplosion";
  MovesEnum2["Mistyterrain"] = "mistyterrain";
  MovesEnum2["Moonblast"] = "moonblast";
  MovesEnum2["Moongeistbeam"] = "moongeistbeam";
  MovesEnum2["Moonlight"] = "moonlight";
  MovesEnum2["Morningsun"] = "morningsun";
  MovesEnum2["Mountaingale"] = "mountaingale";
  MovesEnum2["Mudbomb"] = "mudbomb";
  MovesEnum2["Muddywater"] = "muddywater";
  MovesEnum2["Mudshot"] = "mudshot";
  MovesEnum2["Mudslap"] = "mudslap";
  MovesEnum2["Mudsport"] = "mudsport";
  MovesEnum2["Multiattack"] = "multiattack";
  MovesEnum2["Mysticalfire"] = "mysticalfire";
  MovesEnum2["Mysticalpower"] = "mysticalpower";
  MovesEnum2["Nastyplot"] = "nastyplot";
  MovesEnum2["Naturalgift"] = "naturalgift";
  MovesEnum2["Naturepower"] = "naturepower";
  MovesEnum2["Naturesmadness"] = "naturesmadness";
  MovesEnum2["Needlearm"] = "needlearm";
  MovesEnum2["Neverendingnightmare"] = "neverendingnightmare";
  MovesEnum2["Nightdaze"] = "nightdaze";
  MovesEnum2["Nightmare"] = "nightmare";
  MovesEnum2["Nightshade"] = "nightshade";
  MovesEnum2["Nightslash"] = "nightslash";
  MovesEnum2["Nobleroar"] = "nobleroar";
  MovesEnum2["Noretreat"] = "noretreat";
  MovesEnum2["Nuzzle"] = "nuzzle";
  MovesEnum2["Oblivionwing"] = "oblivionwing";
  MovesEnum2["Obstruct"] = "obstruct";
  MovesEnum2["Oceanicoperetta"] = "oceanicoperetta";
  MovesEnum2["Octazooka"] = "octazooka";
  MovesEnum2["Octolock"] = "octolock";
  MovesEnum2["Odorsleuth"] = "odorsleuth";
  MovesEnum2["Ominouswind"] = "ominouswind";
  MovesEnum2["Originpulse"] = "originpulse";
  MovesEnum2["Outrage"] = "outrage";
  MovesEnum2["Overdrive"] = "overdrive";
  MovesEnum2["Overheat"] = "overheat";
  MovesEnum2["Painsplit"] = "painsplit";
  MovesEnum2["Paleowave"] = "paleowave";
  MovesEnum2["Paraboliccharge"] = "paraboliccharge";
  MovesEnum2["Partingshot"] = "partingshot";
  MovesEnum2["Payback"] = "payback";
  MovesEnum2["Payday"] = "payday";
  MovesEnum2["Peck"] = "peck";
  MovesEnum2["Perishsong"] = "perishsong";
  MovesEnum2["Petalblizzard"] = "petalblizzard";
  MovesEnum2["Petaldance"] = "petaldance";
  MovesEnum2["Phantomforce"] = "phantomforce";
  MovesEnum2["Photongeyser"] = "photongeyser";
  MovesEnum2["Pikapapow"] = "pikapapow";
  MovesEnum2["Pinmissile"] = "pinmissile";
  MovesEnum2["Plasmafists"] = "plasmafists";
  MovesEnum2["Playnice"] = "playnice";
  MovesEnum2["Playrough"] = "playrough";
  MovesEnum2["Pluck"] = "pluck";
  MovesEnum2["Poisonfang"] = "poisonfang";
  MovesEnum2["Poisongas"] = "poisongas";
  MovesEnum2["Poisonjab"] = "poisonjab";
  MovesEnum2["Poisonpowder"] = "poisonpowder";
  MovesEnum2["Poisonsting"] = "poisonsting";
  MovesEnum2["Poisontail"] = "poisontail";
  MovesEnum2["Pollenpuff"] = "pollenpuff";
  MovesEnum2["Poltergeist"] = "poltergeist";
  MovesEnum2["Pound"] = "pound";
  MovesEnum2["Powder"] = "powder";
  MovesEnum2["Powdersnow"] = "powdersnow";
  MovesEnum2["Powergem"] = "powergem";
  MovesEnum2["Powershift"] = "powershift";
  MovesEnum2["Powersplit"] = "powersplit";
  MovesEnum2["Powerswap"] = "powerswap";
  MovesEnum2["Powertrick"] = "powertrick";
  MovesEnum2["Powertrip"] = "powertrip";
  MovesEnum2["Poweruppunch"] = "poweruppunch";
  MovesEnum2["Powerwhip"] = "powerwhip";
  MovesEnum2["Precipiceblades"] = "precipiceblades";
  MovesEnum2["Present"] = "present";
  MovesEnum2["Prismaticlaser"] = "prismaticlaser";
  MovesEnum2["Protect"] = "protect";
  MovesEnum2["Psybeam"] = "psybeam";
  MovesEnum2["Psychic"] = "psychic";
  MovesEnum2["Psychicfangs"] = "psychicfangs";
  MovesEnum2["Psychicterrain"] = "psychicterrain";
  MovesEnum2["Psychoboost"] = "psychoboost";
  MovesEnum2["Psychocut"] = "psychocut";
  MovesEnum2["Psychoshift"] = "psychoshift";
  MovesEnum2["Psychup"] = "psychup";
  MovesEnum2["Psyshieldbash"] = "psyshieldbash";
  MovesEnum2["Psyshock"] = "psyshock";
  MovesEnum2["Psystrike"] = "psystrike";
  MovesEnum2["Psywave"] = "psywave";
  MovesEnum2["Pulverizingpancake"] = "pulverizingpancake";
  MovesEnum2["Punishment"] = "punishment";
  MovesEnum2["Purify"] = "purify";
  MovesEnum2["Pursuit"] = "pursuit";
  MovesEnum2["Pyroball"] = "pyroball";
  MovesEnum2["Quash"] = "quash";
  MovesEnum2["Quickattack"] = "quickattack";
  MovesEnum2["Quickguard"] = "quickguard";
  MovesEnum2["Quiverdance"] = "quiverdance";
  MovesEnum2["Rage"] = "rage";
  MovesEnum2["Ragepowder"] = "ragepowder";
  MovesEnum2["Ragingfury"] = "ragingfury";
  MovesEnum2["Raindance"] = "raindance";
  MovesEnum2["Rapidspin"] = "rapidspin";
  MovesEnum2["Razorleaf"] = "razorleaf";
  MovesEnum2["Razorshell"] = "razorshell";
  MovesEnum2["Razorwind"] = "razorwind";
  MovesEnum2["Recover"] = "recover";
  MovesEnum2["Recycle"] = "recycle";
  MovesEnum2["Reflect"] = "reflect";
  MovesEnum2["Reflecttype"] = "reflecttype";
  MovesEnum2["Refresh"] = "refresh";
  MovesEnum2["Relicsong"] = "relicsong";
  MovesEnum2["Rest"] = "rest";
  MovesEnum2["Retaliate"] = "retaliate";
  MovesEnum2["Return"] = "return";
  MovesEnum2["Revelationdance"] = "revelationdance";
  MovesEnum2["Revenge"] = "revenge";
  MovesEnum2["Reversal"] = "reversal";
  MovesEnum2["Risingvoltage"] = "risingvoltage";
  MovesEnum2["Roar"] = "roar";
  MovesEnum2["Roaroftime"] = "roaroftime";
  MovesEnum2["Rockblast"] = "rockblast";
  MovesEnum2["Rockclimb"] = "rockclimb";
  MovesEnum2["Rockpolish"] = "rockpolish";
  MovesEnum2["Rockslide"] = "rockslide";
  MovesEnum2["Rocksmash"] = "rocksmash";
  MovesEnum2["Rockthrow"] = "rockthrow";
  MovesEnum2["Rocktomb"] = "rocktomb";
  MovesEnum2["Rockwrecker"] = "rockwrecker";
  MovesEnum2["Roleplay"] = "roleplay";
  MovesEnum2["Rollingkick"] = "rollingkick";
  MovesEnum2["Rollout"] = "rollout";
  MovesEnum2["Roost"] = "roost";
  MovesEnum2["Rototiller"] = "rototiller";
  MovesEnum2["Round"] = "round";
  MovesEnum2["Sacredfire"] = "sacredfire";
  MovesEnum2["Sacredsword"] = "sacredsword";
  MovesEnum2["Safeguard"] = "safeguard";
  MovesEnum2["Sandattack"] = "sandattack";
  MovesEnum2["Sandsearstorm"] = "sandsearstorm";
  MovesEnum2["Sandstorm"] = "sandstorm";
  MovesEnum2["Sandtomb"] = "sandtomb";
  MovesEnum2["Sappyseed"] = "sappyseed";
  MovesEnum2["Savagespinout"] = "savagespinout";
  MovesEnum2["Scald"] = "scald";
  MovesEnum2["Scaleshot"] = "scaleshot";
  MovesEnum2["Scaryface"] = "scaryface";
  MovesEnum2["Scorchingsands"] = "scorchingsands";
  MovesEnum2["Scratch"] = "scratch";
  MovesEnum2["Screech"] = "screech";
  MovesEnum2["Searingshot"] = "searingshot";
  MovesEnum2["Searingsunrazesmash"] = "searingsunrazesmash";
  MovesEnum2["Secretpower"] = "secretpower";
  MovesEnum2["Secretsword"] = "secretsword";
  MovesEnum2["Seedbomb"] = "seedbomb";
  MovesEnum2["Seedflare"] = "seedflare";
  MovesEnum2["Seismictoss"] = "seismictoss";
  MovesEnum2["Selfdestruct"] = "selfdestruct";
  MovesEnum2["Shadowball"] = "shadowball";
  MovesEnum2["Shadowbone"] = "shadowbone";
  MovesEnum2["Shadowclaw"] = "shadowclaw";
  MovesEnum2["Shadowforce"] = "shadowforce";
  MovesEnum2["Shadowpunch"] = "shadowpunch";
  MovesEnum2["Shadowsneak"] = "shadowsneak";
  MovesEnum2["Shadowstrike"] = "shadowstrike";
  MovesEnum2["Sharpen"] = "sharpen";
  MovesEnum2["Shatteredpsyche"] = "shatteredpsyche";
  MovesEnum2["Sheercold"] = "sheercold";
  MovesEnum2["Shellsidearm"] = "shellsidearm";
  MovesEnum2["Shellsmash"] = "shellsmash";
  MovesEnum2["Shelltrap"] = "shelltrap";
  MovesEnum2["Shelter"] = "shelter";
  MovesEnum2["Shiftgear"] = "shiftgear";
  MovesEnum2["Shockwave"] = "shockwave";
  MovesEnum2["Shoreup"] = "shoreup";
  MovesEnum2["Signalbeam"] = "signalbeam";
  MovesEnum2["Silverwind"] = "silverwind";
  MovesEnum2["Simplebeam"] = "simplebeam";
  MovesEnum2["Sing"] = "sing";
  MovesEnum2["Sinisterarrowraid"] = "sinisterarrowraid";
  MovesEnum2["Sizzlyslide"] = "sizzlyslide";
  MovesEnum2["Sketch"] = "sketch";
  MovesEnum2["Skillswap"] = "skillswap";
  MovesEnum2["Skittersmack"] = "skittersmack";
  MovesEnum2["Skullbash"] = "skullbash";
  MovesEnum2["Skyattack"] = "skyattack";
  MovesEnum2["Skydrop"] = "skydrop";
  MovesEnum2["Skyuppercut"] = "skyuppercut";
  MovesEnum2["Slackoff"] = "slackoff";
  MovesEnum2["Slam"] = "slam";
  MovesEnum2["Slash"] = "slash";
  MovesEnum2["Sleeppowder"] = "sleeppowder";
  MovesEnum2["Sleeptalk"] = "sleeptalk";
  MovesEnum2["Sludge"] = "sludge";
  MovesEnum2["Sludgebomb"] = "sludgebomb";
  MovesEnum2["Sludgewave"] = "sludgewave";
  MovesEnum2["Smackdown"] = "smackdown";
  MovesEnum2["Smartstrike"] = "smartstrike";
  MovesEnum2["Smellingsalts"] = "smellingsalts";
  MovesEnum2["Smog"] = "smog";
  MovesEnum2["Smokescreen"] = "smokescreen";
  MovesEnum2["Snaptrap"] = "snaptrap";
  MovesEnum2["Snarl"] = "snarl";
  MovesEnum2["Snatch"] = "snatch";
  MovesEnum2["Snipeshot"] = "snipeshot";
  MovesEnum2["Snore"] = "snore";
  MovesEnum2["Soak"] = "soak";
  MovesEnum2["Softboiled"] = "softboiled";
  MovesEnum2["Solarbeam"] = "solarbeam";
  MovesEnum2["Solarblade"] = "solarblade";
  MovesEnum2["Sonicboom"] = "sonicboom";
  MovesEnum2["Soulstealing7starstrike"] = "soulstealing7starstrike";
  MovesEnum2["Spacialrend"] = "spacialrend";
  MovesEnum2["Spark"] = "spark";
  MovesEnum2["Sparklingaria"] = "sparklingaria";
  MovesEnum2["Sparklyswirl"] = "sparklyswirl";
  MovesEnum2["Spectralthief"] = "spectralthief";
  MovesEnum2["Speedswap"] = "speedswap";
  MovesEnum2["Spiderweb"] = "spiderweb";
  MovesEnum2["Spikecannon"] = "spikecannon";
  MovesEnum2["Spikes"] = "spikes";
  MovesEnum2["Spikyshield"] = "spikyshield";
  MovesEnum2["Spiritbreak"] = "spiritbreak";
  MovesEnum2["Spiritshackle"] = "spiritshackle";
  MovesEnum2["Spite"] = "spite";
  MovesEnum2["Spitup"] = "spitup";
  MovesEnum2["Splash"] = "splash";
  MovesEnum2["Splinteredstormshards"] = "splinteredstormshards";
  MovesEnum2["Splishysplash"] = "splishysplash";
  MovesEnum2["Spore"] = "spore";
  MovesEnum2["Spotlight"] = "spotlight";
  MovesEnum2["Springtidestorm"] = "springtidestorm";
  MovesEnum2["Stealthrock"] = "stealthrock";
  MovesEnum2["Steameruption"] = "steameruption";
  MovesEnum2["Steamroller"] = "steamroller";
  MovesEnum2["Steelbeam"] = "steelbeam";
  MovesEnum2["Steelroller"] = "steelroller";
  MovesEnum2["Steelwing"] = "steelwing";
  MovesEnum2["Stickyweb"] = "stickyweb";
  MovesEnum2["Stockpile"] = "stockpile";
  MovesEnum2["Stokedsparksurfer"] = "stokedsparksurfer";
  MovesEnum2["Stomp"] = "stomp";
  MovesEnum2["Stompingtantrum"] = "stompingtantrum";
  MovesEnum2["Stoneaxe"] = "stoneaxe";
  MovesEnum2["Stoneedge"] = "stoneedge";
  MovesEnum2["Storedpower"] = "storedpower";
  MovesEnum2["Stormthrow"] = "stormthrow";
  MovesEnum2["Strangesteam"] = "strangesteam";
  MovesEnum2["Strength"] = "strength";
  MovesEnum2["Strengthsap"] = "strengthsap";
  MovesEnum2["Stringshot"] = "stringshot";
  MovesEnum2["Struggle"] = "struggle";
  MovesEnum2["Strugglebug"] = "strugglebug";
  MovesEnum2["Stuffcheeks"] = "stuffcheeks";
  MovesEnum2["Stunspore"] = "stunspore";
  MovesEnum2["Submission"] = "submission";
  MovesEnum2["Substitute"] = "substitute";
  MovesEnum2["Subzeroslammer"] = "subzeroslammer";
  MovesEnum2["Suckerpunch"] = "suckerpunch";
  MovesEnum2["Sunnyday"] = "sunnyday";
  MovesEnum2["Sunsteelstrike"] = "sunsteelstrike";
  MovesEnum2["Superfang"] = "superfang";
  MovesEnum2["Superpower"] = "superpower";
  MovesEnum2["Supersonic"] = "supersonic";
  MovesEnum2["Supersonicskystrike"] = "supersonicskystrike";
  MovesEnum2["Surf"] = "surf";
  MovesEnum2["Surgingstrikes"] = "surgingstrikes";
  MovesEnum2["Swagger"] = "swagger";
  MovesEnum2["Swallow"] = "swallow";
  MovesEnum2["Sweetkiss"] = "sweetkiss";
  MovesEnum2["Sweetscent"] = "sweetscent";
  MovesEnum2["Swift"] = "swift";
  MovesEnum2["Switcheroo"] = "switcheroo";
  MovesEnum2["Swordsdance"] = "swordsdance";
  MovesEnum2["Synchronoise"] = "synchronoise";
  MovesEnum2["Synthesis"] = "synthesis";
  MovesEnum2["Tackle"] = "tackle";
  MovesEnum2["Tailglow"] = "tailglow";
  MovesEnum2["Tailslap"] = "tailslap";
  MovesEnum2["Tailwhip"] = "tailwhip";
  MovesEnum2["Tailwind"] = "tailwind";
  MovesEnum2["Takedown"] = "takedown";
  MovesEnum2["Takeheart"] = "takeheart";
  MovesEnum2["Tarshot"] = "tarshot";
  MovesEnum2["Taunt"] = "taunt";
  MovesEnum2["Tearfullook"] = "tearfullook";
  MovesEnum2["Teatime"] = "teatime";
  MovesEnum2["Technoblast"] = "technoblast";
  MovesEnum2["Tectonicrage"] = "tectonicrage";
  MovesEnum2["Teeterdance"] = "teeterdance";
  MovesEnum2["Telekinesis"] = "telekinesis";
  MovesEnum2["Teleport"] = "teleport";
  MovesEnum2["Tenmillionvoltthunderbolt"] = "tenmillionvoltthunderbolt";
  MovesEnum2["Terrainpulse"] = "terrainpulse";
  MovesEnum2["Thief"] = "thief";
  MovesEnum2["Thousandarrows"] = "thousandarrows";
  MovesEnum2["Thousandwaves"] = "thousandwaves";
  MovesEnum2["Thrash"] = "thrash";
  MovesEnum2["Throatchop"] = "throatchop";
  MovesEnum2["Thunder"] = "thunder";
  MovesEnum2["Thunderbolt"] = "thunderbolt";
  MovesEnum2["Thundercage"] = "thundercage";
  MovesEnum2["Thunderfang"] = "thunderfang";
  MovesEnum2["Thunderouskick"] = "thunderouskick";
  MovesEnum2["Thunderpunch"] = "thunderpunch";
  MovesEnum2["Thundershock"] = "thundershock";
  MovesEnum2["Thunderwave"] = "thunderwave";
  MovesEnum2["Tickle"] = "tickle";
  MovesEnum2["Topsyturvy"] = "topsyturvy";
  MovesEnum2["Torment"] = "torment";
  MovesEnum2["Toxic"] = "toxic";
  MovesEnum2["Toxicspikes"] = "toxicspikes";
  MovesEnum2["Toxicthread"] = "toxicthread";
  MovesEnum2["Transform"] = "transform";
  MovesEnum2["Triattack"] = "triattack";
  MovesEnum2["Trick"] = "trick";
  MovesEnum2["Trickortreat"] = "trickortreat";
  MovesEnum2["Trickroom"] = "trickroom";
  MovesEnum2["Triplearrows"] = "triplearrows";
  MovesEnum2["Tripleaxel"] = "tripleaxel";
  MovesEnum2["Triplekick"] = "triplekick";
  MovesEnum2["Tropkick"] = "tropkick";
  MovesEnum2["Trumpcard"] = "trumpcard";
  MovesEnum2["Twineedle"] = "twineedle";
  MovesEnum2["Twinkletackle"] = "twinkletackle";
  MovesEnum2["Twister"] = "twister";
  MovesEnum2["Uproar"] = "uproar";
  MovesEnum2["Uturn"] = "uturn";
  MovesEnum2["Vacuumwave"] = "vacuumwave";
  MovesEnum2["Vcreate"] = "vcreate";
  MovesEnum2["Veeveevolley"] = "veeveevolley";
  MovesEnum2["Venomdrench"] = "venomdrench";
  MovesEnum2["Venoshock"] = "venoshock";
  MovesEnum2["Victorydance"] = "victorydance";
  MovesEnum2["Vinewhip"] = "vinewhip";
  MovesEnum2["Visegrip"] = "visegrip";
  MovesEnum2["Vitalthrow"] = "vitalthrow";
  MovesEnum2["Voltswitch"] = "voltswitch";
  MovesEnum2["Volttackle"] = "volttackle";
  MovesEnum2["Wakeupslap"] = "wakeupslap";
  MovesEnum2["Waterfall"] = "waterfall";
  MovesEnum2["Watergun"] = "watergun";
  MovesEnum2["Waterpledge"] = "waterpledge";
  MovesEnum2["Waterpulse"] = "waterpulse";
  MovesEnum2["Watershuriken"] = "watershuriken";
  MovesEnum2["Watersport"] = "watersport";
  MovesEnum2["Waterspout"] = "waterspout";
  MovesEnum2["Wavecrash"] = "wavecrash";
  MovesEnum2["Weatherball"] = "weatherball";
  MovesEnum2["Whirlpool"] = "whirlpool";
  MovesEnum2["Whirlwind"] = "whirlwind";
  MovesEnum2["Wickedblow"] = "wickedblow";
  MovesEnum2["Wideguard"] = "wideguard";
  MovesEnum2["Wildboltstorm"] = "wildboltstorm";
  MovesEnum2["Wildcharge"] = "wildcharge";
  MovesEnum2["Willowisp"] = "willowisp";
  MovesEnum2["Wingattack"] = "wingattack";
  MovesEnum2["Wish"] = "wish";
  MovesEnum2["Withdraw"] = "withdraw";
  MovesEnum2["Wonderroom"] = "wonderroom";
  MovesEnum2["Woodhammer"] = "woodhammer";
  MovesEnum2["Workup"] = "workup";
  MovesEnum2["Worryseed"] = "worryseed";
  MovesEnum2["Wrap"] = "wrap";
  MovesEnum2["Wringout"] = "wringout";
  MovesEnum2["Xscissor"] = "xscissor";
  MovesEnum2["Yawn"] = "yawn";
  MovesEnum2["Zapcannon"] = "zapcannon";
  MovesEnum2["Zenheadbutt"] = "zenheadbutt";
  MovesEnum2["Zingzap"] = "zingzap";
  MovesEnum2["Zippyzap"] = "zippyzap";
  return MovesEnum2;
})(MovesEnum || {});
var PokemonEnum = /* @__PURE__ */ ((PokemonEnum2) => {
  PokemonEnum2["Abomasnow"] = "abomasnow";
  PokemonEnum2["Abomasnowmega"] = "abomasnowmega";
  PokemonEnum2["Abra"] = "abra";
  PokemonEnum2["Absol"] = "absol";
  PokemonEnum2["Absolmega"] = "absolmega";
  PokemonEnum2["Accelgor"] = "accelgor";
  PokemonEnum2["Aegislash"] = "aegislash";
  PokemonEnum2["Aegislashblade"] = "aegislashblade";
  PokemonEnum2["Aerodactyl"] = "aerodactyl";
  PokemonEnum2["Aerodactylmega"] = "aerodactylmega";
  PokemonEnum2["Aggron"] = "aggron";
  PokemonEnum2["Aggronmega"] = "aggronmega";
  PokemonEnum2["Aipom"] = "aipom";
  PokemonEnum2["Alakazam"] = "alakazam";
  PokemonEnum2["Alakazammega"] = "alakazammega";
  PokemonEnum2["Alcremie"] = "alcremie";
  PokemonEnum2["Alcremiegmax"] = "alcremiegmax";
  PokemonEnum2["Alomomola"] = "alomomola";
  PokemonEnum2["Altaria"] = "altaria";
  PokemonEnum2["Altariamega"] = "altariamega";
  PokemonEnum2["Amaura"] = "amaura";
  PokemonEnum2["Ambipom"] = "ambipom";
  PokemonEnum2["Amoonguss"] = "amoonguss";
  PokemonEnum2["Ampharos"] = "ampharos";
  PokemonEnum2["Ampharosmega"] = "ampharosmega";
  PokemonEnum2["Anorith"] = "anorith";
  PokemonEnum2["Appletun"] = "appletun";
  PokemonEnum2["Appletungmax"] = "appletungmax";
  PokemonEnum2["Applin"] = "applin";
  PokemonEnum2["Araquanid"] = "araquanid";
  PokemonEnum2["Araquanidtotem"] = "araquanidtotem";
  PokemonEnum2["Arbok"] = "arbok";
  PokemonEnum2["Arcanine"] = "arcanine";
  PokemonEnum2["Arcaninehisui"] = "arcaninehisui";
  PokemonEnum2["Arceus"] = "arceus";
  PokemonEnum2["Arceusbug"] = "arceusbug";
  PokemonEnum2["Arceusdark"] = "arceusdark";
  PokemonEnum2["Arceusdragon"] = "arceusdragon";
  PokemonEnum2["Arceuselectric"] = "arceuselectric";
  PokemonEnum2["Arceusfairy"] = "arceusfairy";
  PokemonEnum2["Arceusfighting"] = "arceusfighting";
  PokemonEnum2["Arceusfire"] = "arceusfire";
  PokemonEnum2["Arceusflying"] = "arceusflying";
  PokemonEnum2["Arceusghost"] = "arceusghost";
  PokemonEnum2["Arceusgrass"] = "arceusgrass";
  PokemonEnum2["Arceusground"] = "arceusground";
  PokemonEnum2["Arceusice"] = "arceusice";
  PokemonEnum2["Arceuslegend"] = "arceuslegend";
  PokemonEnum2["Arceuspoison"] = "arceuspoison";
  PokemonEnum2["Arceuspsychic"] = "arceuspsychic";
  PokemonEnum2["Arceusrock"] = "arceusrock";
  PokemonEnum2["Arceussteel"] = "arceussteel";
  PokemonEnum2["Arceuswater"] = "arceuswater";
  PokemonEnum2["Archen"] = "archen";
  PokemonEnum2["Archeops"] = "archeops";
  PokemonEnum2["Arctovish"] = "arctovish";
  PokemonEnum2["Arctozolt"] = "arctozolt";
  PokemonEnum2["Argalis"] = "argalis";
  PokemonEnum2["Arghonaut"] = "arghonaut";
  PokemonEnum2["Ariados"] = "ariados";
  PokemonEnum2["Armaldo"] = "armaldo";
  PokemonEnum2["Aromatisse"] = "aromatisse";
  PokemonEnum2["Aron"] = "aron";
  PokemonEnum2["Arrokuda"] = "arrokuda";
  PokemonEnum2["Articuno"] = "articuno";
  PokemonEnum2["Articunogalar"] = "articunogalar";
  PokemonEnum2["Astrolotl"] = "astrolotl";
  PokemonEnum2["Audino"] = "audino";
  PokemonEnum2["Audinomega"] = "audinomega";
  PokemonEnum2["Aurorus"] = "aurorus";
  PokemonEnum2["Aurumoth"] = "aurumoth";
  PokemonEnum2["Avalugg"] = "avalugg";
  PokemonEnum2["Avalugghisui"] = "avalugghisui";
  PokemonEnum2["Axew"] = "axew";
  PokemonEnum2["Azelf"] = "azelf";
  PokemonEnum2["Azumarill"] = "azumarill";
  PokemonEnum2["Azurill"] = "azurill";
  PokemonEnum2["Bagon"] = "bagon";
  PokemonEnum2["Baltoy"] = "baltoy";
  PokemonEnum2["Banette"] = "banette";
  PokemonEnum2["Banettemega"] = "banettemega";
  PokemonEnum2["Barbaracle"] = "barbaracle";
  PokemonEnum2["Barboach"] = "barboach";
  PokemonEnum2["Barraskewda"] = "barraskewda";
  PokemonEnum2["Basculegion"] = "basculegion";
  PokemonEnum2["Basculegionf"] = "basculegionf";
  PokemonEnum2["Basculin"] = "basculin";
  PokemonEnum2["Basculinbluestriped"] = "basculinbluestriped";
  PokemonEnum2["Basculinwhitestriped"] = "basculinwhitestriped";
  PokemonEnum2["Bastiodon"] = "bastiodon";
  PokemonEnum2["Bayleef"] = "bayleef";
  PokemonEnum2["Beartic"] = "beartic";
  PokemonEnum2["Beautifly"] = "beautifly";
  PokemonEnum2["Beedrill"] = "beedrill";
  PokemonEnum2["Beedrillmega"] = "beedrillmega";
  PokemonEnum2["Beheeyem"] = "beheeyem";
  PokemonEnum2["Beldum"] = "beldum";
  PokemonEnum2["Bellossom"] = "bellossom";
  PokemonEnum2["Bellsprout"] = "bellsprout";
  PokemonEnum2["Bergmite"] = "bergmite";
  PokemonEnum2["Bewear"] = "bewear";
  PokemonEnum2["Bibarel"] = "bibarel";
  PokemonEnum2["Bidoof"] = "bidoof";
  PokemonEnum2["Binacle"] = "binacle";
  PokemonEnum2["Bisharp"] = "bisharp";
  PokemonEnum2["Blacephalon"] = "blacephalon";
  PokemonEnum2["Blastoise"] = "blastoise";
  PokemonEnum2["Blastoisegmax"] = "blastoisegmax";
  PokemonEnum2["Blastoisemega"] = "blastoisemega";
  PokemonEnum2["Blaziken"] = "blaziken";
  PokemonEnum2["Blazikenmega"] = "blazikenmega";
  PokemonEnum2["Blipbug"] = "blipbug";
  PokemonEnum2["Blissey"] = "blissey";
  PokemonEnum2["Blitzle"] = "blitzle";
  PokemonEnum2["Boldore"] = "boldore";
  PokemonEnum2["Boltund"] = "boltund";
  PokemonEnum2["Bonsly"] = "bonsly";
  PokemonEnum2["Bouffalant"] = "bouffalant";
  PokemonEnum2["Bounsweet"] = "bounsweet";
  PokemonEnum2["Braixen"] = "braixen";
  PokemonEnum2["Brattler"] = "brattler";
  PokemonEnum2["Braviary"] = "braviary";
  PokemonEnum2["Braviaryhisui"] = "braviaryhisui";
  PokemonEnum2["Breezi"] = "breezi";
  PokemonEnum2["Breloom"] = "breloom";
  PokemonEnum2["Brionne"] = "brionne";
  PokemonEnum2["Bronzong"] = "bronzong";
  PokemonEnum2["Bronzor"] = "bronzor";
  PokemonEnum2["Bruxish"] = "bruxish";
  PokemonEnum2["Budew"] = "budew";
  PokemonEnum2["Buizel"] = "buizel";
  PokemonEnum2["Bulbasaur"] = "bulbasaur";
  PokemonEnum2["Buneary"] = "buneary";
  PokemonEnum2["Bunnelby"] = "bunnelby";
  PokemonEnum2["Burmy"] = "burmy";
  PokemonEnum2["Butterfree"] = "butterfree";
  PokemonEnum2["Butterfreegmax"] = "butterfreegmax";
  PokemonEnum2["Buzzwole"] = "buzzwole";
  PokemonEnum2["Cacnea"] = "cacnea";
  PokemonEnum2["Cacturne"] = "cacturne";
  PokemonEnum2["Caimanoe"] = "caimanoe";
  PokemonEnum2["Calyrex"] = "calyrex";
  PokemonEnum2["Calyrexice"] = "calyrexice";
  PokemonEnum2["Calyrexshadow"] = "calyrexshadow";
  PokemonEnum2["Camerupt"] = "camerupt";
  PokemonEnum2["Cameruptmega"] = "cameruptmega";
  PokemonEnum2["Carbink"] = "carbink";
  PokemonEnum2["Caribolt"] = "caribolt";
  PokemonEnum2["Carkol"] = "carkol";
  PokemonEnum2["Carnivine"] = "carnivine";
  PokemonEnum2["Carracosta"] = "carracosta";
  PokemonEnum2["Carvanha"] = "carvanha";
  PokemonEnum2["Cascoon"] = "cascoon";
  PokemonEnum2["Castform"] = "castform";
  PokemonEnum2["Castformrainy"] = "castformrainy";
  PokemonEnum2["Castformsnowy"] = "castformsnowy";
  PokemonEnum2["Castformsunny"] = "castformsunny";
  PokemonEnum2["Caterpie"] = "caterpie";
  PokemonEnum2["Cawdet"] = "cawdet";
  PokemonEnum2["Cawmodore"] = "cawmodore";
  PokemonEnum2["Celebi"] = "celebi";
  PokemonEnum2["Celesteela"] = "celesteela";
  PokemonEnum2["Centiskorch"] = "centiskorch";
  PokemonEnum2["Centiskorchgmax"] = "centiskorchgmax";
  PokemonEnum2["Chandelure"] = "chandelure";
  PokemonEnum2["Chansey"] = "chansey";
  PokemonEnum2["Charizard"] = "charizard";
  PokemonEnum2["Charizardgmax"] = "charizardgmax";
  PokemonEnum2["Charizardmegax"] = "charizardmegax";
  PokemonEnum2["Charizardmegay"] = "charizardmegay";
  PokemonEnum2["Charjabug"] = "charjabug";
  PokemonEnum2["Charmander"] = "charmander";
  PokemonEnum2["Charmeleon"] = "charmeleon";
  PokemonEnum2["Chatot"] = "chatot";
  PokemonEnum2["Cherrim"] = "cherrim";
  PokemonEnum2["Cherrimsunshine"] = "cherrimsunshine";
  PokemonEnum2["Cherubi"] = "cherubi";
  PokemonEnum2["Chesnaught"] = "chesnaught";
  PokemonEnum2["Chespin"] = "chespin";
  PokemonEnum2["Chewtle"] = "chewtle";
  PokemonEnum2["Chikorita"] = "chikorita";
  PokemonEnum2["Chimchar"] = "chimchar";
  PokemonEnum2["Chimecho"] = "chimecho";
  PokemonEnum2["Chinchou"] = "chinchou";
  PokemonEnum2["Chingling"] = "chingling";
  PokemonEnum2["Chromera"] = "chromera";
  PokemonEnum2["Cinccino"] = "cinccino";
  PokemonEnum2["Cinderace"] = "cinderace";
  PokemonEnum2["Cinderacegmax"] = "cinderacegmax";
  PokemonEnum2["Clamperl"] = "clamperl";
  PokemonEnum2["Clauncher"] = "clauncher";
  PokemonEnum2["Clawitzer"] = "clawitzer";
  PokemonEnum2["Claydol"] = "claydol";
  PokemonEnum2["Clefable"] = "clefable";
  PokemonEnum2["Clefairy"] = "clefairy";
  PokemonEnum2["Cleffa"] = "cleffa";
  PokemonEnum2["Clobbopus"] = "clobbopus";
  PokemonEnum2["Cloyster"] = "cloyster";
  PokemonEnum2["Coalossal"] = "coalossal";
  PokemonEnum2["Coalossalgmax"] = "coalossalgmax";
  PokemonEnum2["Cobalion"] = "cobalion";
  PokemonEnum2["Cofagrigus"] = "cofagrigus";
  PokemonEnum2["Colossoil"] = "colossoil";
  PokemonEnum2["Combee"] = "combee";
  PokemonEnum2["Combusken"] = "combusken";
  PokemonEnum2["Comfey"] = "comfey";
  PokemonEnum2["Conkeldurr"] = "conkeldurr";
  PokemonEnum2["Copperajah"] = "copperajah";
  PokemonEnum2["Copperajahgmax"] = "copperajahgmax";
  PokemonEnum2["Coribalis"] = "coribalis";
  PokemonEnum2["Corphish"] = "corphish";
  PokemonEnum2["Corsola"] = "corsola";
  PokemonEnum2["Corsolagalar"] = "corsolagalar";
  PokemonEnum2["Corviknight"] = "corviknight";
  PokemonEnum2["Corviknightgmax"] = "corviknightgmax";
  PokemonEnum2["Corvisquire"] = "corvisquire";
  PokemonEnum2["Cosmoem"] = "cosmoem";
  PokemonEnum2["Cosmog"] = "cosmog";
  PokemonEnum2["Cottonee"] = "cottonee";
  PokemonEnum2["Crabominable"] = "crabominable";
  PokemonEnum2["Crabrawler"] = "crabrawler";
  PokemonEnum2["Cradily"] = "cradily";
  PokemonEnum2["Cramorant"] = "cramorant";
  PokemonEnum2["Cramorantgorging"] = "cramorantgorging";
  PokemonEnum2["Cramorantgulping"] = "cramorantgulping";
  PokemonEnum2["Cranidos"] = "cranidos";
  PokemonEnum2["Crawdaunt"] = "crawdaunt";
  PokemonEnum2["Cresselia"] = "cresselia";
  PokemonEnum2["Croagunk"] = "croagunk";
  PokemonEnum2["Crobat"] = "crobat";
  PokemonEnum2["Croconaw"] = "croconaw";
  PokemonEnum2["Crucibelle"] = "crucibelle";
  PokemonEnum2["Crucibellemega"] = "crucibellemega";
  PokemonEnum2["Crustle"] = "crustle";
  PokemonEnum2["Cryogonal"] = "cryogonal";
  PokemonEnum2["Cubchoo"] = "cubchoo";
  PokemonEnum2["Cubone"] = "cubone";
  PokemonEnum2["Cufant"] = "cufant";
  PokemonEnum2["Cupra"] = "cupra";
  PokemonEnum2["Cursola"] = "cursola";
  PokemonEnum2["Cutiefly"] = "cutiefly";
  PokemonEnum2["Cyclohm"] = "cyclohm";
  PokemonEnum2["Cyndaquil"] = "cyndaquil";
  PokemonEnum2["Darkrai"] = "darkrai";
  PokemonEnum2["Darmanitan"] = "darmanitan";
  PokemonEnum2["Darmanitangalar"] = "darmanitangalar";
  PokemonEnum2["Darmanitangalarzen"] = "darmanitangalarzen";
  PokemonEnum2["Darmanitanzen"] = "darmanitanzen";
  PokemonEnum2["Dartrix"] = "dartrix";
  PokemonEnum2["Darumaka"] = "darumaka";
  PokemonEnum2["Darumakagalar"] = "darumakagalar";
  PokemonEnum2["Decidueye"] = "decidueye";
  PokemonEnum2["Decidueyehisui"] = "decidueyehisui";
  PokemonEnum2["Dedenne"] = "dedenne";
  PokemonEnum2["Deerling"] = "deerling";
  PokemonEnum2["Deino"] = "deino";
  PokemonEnum2["Delcatty"] = "delcatty";
  PokemonEnum2["Delibird"] = "delibird";
  PokemonEnum2["Delphox"] = "delphox";
  PokemonEnum2["Deoxys"] = "deoxys";
  PokemonEnum2["Deoxysattack"] = "deoxysattack";
  PokemonEnum2["Deoxysdefense"] = "deoxysdefense";
  PokemonEnum2["Deoxysspeed"] = "deoxysspeed";
  PokemonEnum2["Dewgong"] = "dewgong";
  PokemonEnum2["Dewott"] = "dewott";
  PokemonEnum2["Dewpider"] = "dewpider";
  PokemonEnum2["Dhelmise"] = "dhelmise";
  PokemonEnum2["Dialga"] = "dialga";
  PokemonEnum2["Dialgaorigin"] = "dialgaorigin";
  PokemonEnum2["Diancie"] = "diancie";
  PokemonEnum2["Dianciemega"] = "dianciemega";
  PokemonEnum2["Diggersby"] = "diggersby";
  PokemonEnum2["Diglett"] = "diglett";
  PokemonEnum2["Diglettalola"] = "diglettalola";
  PokemonEnum2["Ditto"] = "ditto";
  PokemonEnum2["Dodrio"] = "dodrio";
  PokemonEnum2["Doduo"] = "doduo";
  PokemonEnum2["Donphan"] = "donphan";
  PokemonEnum2["Dorsoil"] = "dorsoil";
  PokemonEnum2["Dottler"] = "dottler";
  PokemonEnum2["Doublade"] = "doublade";
  PokemonEnum2["Dracovish"] = "dracovish";
  PokemonEnum2["Dracozolt"] = "dracozolt";
  PokemonEnum2["Dragalge"] = "dragalge";
  PokemonEnum2["Dragapult"] = "dragapult";
  PokemonEnum2["Dragonair"] = "dragonair";
  PokemonEnum2["Dragonite"] = "dragonite";
  PokemonEnum2["Drakloak"] = "drakloak";
  PokemonEnum2["Drampa"] = "drampa";
  PokemonEnum2["Drapion"] = "drapion";
  PokemonEnum2["Dratini"] = "dratini";
  PokemonEnum2["Drednaw"] = "drednaw";
  PokemonEnum2["Drednawgmax"] = "drednawgmax";
  PokemonEnum2["Dreepy"] = "dreepy";
  PokemonEnum2["Drifblim"] = "drifblim";
  PokemonEnum2["Drifloon"] = "drifloon";
  PokemonEnum2["Drilbur"] = "drilbur";
  PokemonEnum2["Drizzile"] = "drizzile";
  PokemonEnum2["Drowzee"] = "drowzee";
  PokemonEnum2["Druddigon"] = "druddigon";
  PokemonEnum2["Dubwool"] = "dubwool";
  PokemonEnum2["Ducklett"] = "ducklett";
  PokemonEnum2["Dugtrio"] = "dugtrio";
  PokemonEnum2["Dugtrioalola"] = "dugtrioalola";
  PokemonEnum2["Dunsparce"] = "dunsparce";
  PokemonEnum2["Duohm"] = "duohm";
  PokemonEnum2["Duosion"] = "duosion";
  PokemonEnum2["Duraludon"] = "duraludon";
  PokemonEnum2["Duraludongmax"] = "duraludongmax";
  PokemonEnum2["Durant"] = "durant";
  PokemonEnum2["Dusclops"] = "dusclops";
  PokemonEnum2["Dusknoir"] = "dusknoir";
  PokemonEnum2["Duskull"] = "duskull";
  PokemonEnum2["Dustox"] = "dustox";
  PokemonEnum2["Dwebble"] = "dwebble";
  PokemonEnum2["Eelektrik"] = "eelektrik";
  PokemonEnum2["Eelektross"] = "eelektross";
  PokemonEnum2["Eevee"] = "eevee";
  PokemonEnum2["Eeveegmax"] = "eeveegmax";
  PokemonEnum2["Eeveestarter"] = "eeveestarter";
  PokemonEnum2["Eiscue"] = "eiscue";
  PokemonEnum2["Eiscuenoice"] = "eiscuenoice";
  PokemonEnum2["Ekans"] = "ekans";
  PokemonEnum2["Eldegoss"] = "eldegoss";
  PokemonEnum2["Electabuzz"] = "electabuzz";
  PokemonEnum2["Electivire"] = "electivire";
  PokemonEnum2["Electrelk"] = "electrelk";
  PokemonEnum2["Electrike"] = "electrike";
  PokemonEnum2["Electrode"] = "electrode";
  PokemonEnum2["Electrodehisui"] = "electrodehisui";
  PokemonEnum2["Elekid"] = "elekid";
  PokemonEnum2["Elgyem"] = "elgyem";
  PokemonEnum2["Embirch"] = "embirch";
  PokemonEnum2["Emboar"] = "emboar";
  PokemonEnum2["Emolga"] = "emolga";
  PokemonEnum2["Empoleon"] = "empoleon";
  PokemonEnum2["Enamorus"] = "enamorus";
  PokemonEnum2["Enamorustherian"] = "enamorustherian";
  PokemonEnum2["Entei"] = "entei";
  PokemonEnum2["Equilibra"] = "equilibra";
  PokemonEnum2["Escavalier"] = "escavalier";
  PokemonEnum2["Espeon"] = "espeon";
  PokemonEnum2["Espurr"] = "espurr";
  PokemonEnum2["Eternatus"] = "eternatus";
  PokemonEnum2["Eternatuseternamax"] = "eternatuseternamax";
  PokemonEnum2["Excadrill"] = "excadrill";
  PokemonEnum2["Exeggcute"] = "exeggcute";
  PokemonEnum2["Exeggutor"] = "exeggutor";
  PokemonEnum2["Exeggutoralola"] = "exeggutoralola";
  PokemonEnum2["Exploud"] = "exploud";
  PokemonEnum2["Falinks"] = "falinks";
  PokemonEnum2["Farfetchd"] = "farfetchd";
  PokemonEnum2["Farfetchdgalar"] = "farfetchdgalar";
  PokemonEnum2["Fawnifer"] = "fawnifer";
  PokemonEnum2["Fearow"] = "fearow";
  PokemonEnum2["Feebas"] = "feebas";
  PokemonEnum2["Fennekin"] = "fennekin";
  PokemonEnum2["Feraligatr"] = "feraligatr";
  PokemonEnum2["Ferroseed"] = "ferroseed";
  PokemonEnum2["Ferrothorn"] = "ferrothorn";
  PokemonEnum2["Fidgit"] = "fidgit";
  PokemonEnum2["Finneon"] = "finneon";
  PokemonEnum2["Flaaffy"] = "flaaffy";
  PokemonEnum2["Flabebe"] = "flabebe";
  PokemonEnum2["Flapple"] = "flapple";
  PokemonEnum2["Flapplegmax"] = "flapplegmax";
  PokemonEnum2["Flarelm"] = "flarelm";
  PokemonEnum2["Flareon"] = "flareon";
  PokemonEnum2["Fletchinder"] = "fletchinder";
  PokemonEnum2["Fletchling"] = "fletchling";
  PokemonEnum2["Floatoy"] = "floatoy";
  PokemonEnum2["Floatzel"] = "floatzel";
  PokemonEnum2["Floette"] = "floette";
  PokemonEnum2["Floetteeternal"] = "floetteeternal";
  PokemonEnum2["Florges"] = "florges";
  PokemonEnum2["Flygon"] = "flygon";
  PokemonEnum2["Fomantis"] = "fomantis";
  PokemonEnum2["Foongus"] = "foongus";
  PokemonEnum2["Forretress"] = "forretress";
  PokemonEnum2["Fraxure"] = "fraxure";
  PokemonEnum2["Frillish"] = "frillish";
  PokemonEnum2["Frillishfemale"] = "frillishfemale";
  PokemonEnum2["Froakie"] = "froakie";
  PokemonEnum2["Frogadier"] = "frogadier";
  PokemonEnum2["Froslass"] = "froslass";
  PokemonEnum2["Frosmoth"] = "frosmoth";
  PokemonEnum2["Furfrou"] = "furfrou";
  PokemonEnum2["Furret"] = "furret";
  PokemonEnum2["Gabite"] = "gabite";
  PokemonEnum2["Gallade"] = "gallade";
  PokemonEnum2["Gallademega"] = "gallademega";
  PokemonEnum2["Galvantula"] = "galvantula";
  PokemonEnum2["Garbodor"] = "garbodor";
  PokemonEnum2["Garbodorgmax"] = "garbodorgmax";
  PokemonEnum2["Garchomp"] = "garchomp";
  PokemonEnum2["Garchompmega"] = "garchompmega";
  PokemonEnum2["Gardevoir"] = "gardevoir";
  PokemonEnum2["Gardevoirmega"] = "gardevoirmega";
  PokemonEnum2["Gastly"] = "gastly";
  PokemonEnum2["Gastrodon"] = "gastrodon";
  PokemonEnum2["Genesect"] = "genesect";
  PokemonEnum2["Genesectburn"] = "genesectburn";
  PokemonEnum2["Genesectchill"] = "genesectchill";
  PokemonEnum2["Genesectdouse"] = "genesectdouse";
  PokemonEnum2["Genesectshock"] = "genesectshock";
  PokemonEnum2["Gengar"] = "gengar";
  PokemonEnum2["Gengargmax"] = "gengargmax";
  PokemonEnum2["Gengarmega"] = "gengarmega";
  PokemonEnum2["Geodude"] = "geodude";
  PokemonEnum2["Geodudealola"] = "geodudealola";
  PokemonEnum2["Gible"] = "gible";
  PokemonEnum2["Gigalith"] = "gigalith";
  PokemonEnum2["Girafarig"] = "girafarig";
  PokemonEnum2["Giratina"] = "giratina";
  PokemonEnum2["Giratinaorigin"] = "giratinaorigin";
  PokemonEnum2["Glaceon"] = "glaceon";
  PokemonEnum2["Glalie"] = "glalie";
  PokemonEnum2["Glaliemega"] = "glaliemega";
  PokemonEnum2["Glameow"] = "glameow";
  PokemonEnum2["Glastrier"] = "glastrier";
  PokemonEnum2["Gligar"] = "gligar";
  PokemonEnum2["Gliscor"] = "gliscor";
  PokemonEnum2["Gloom"] = "gloom";
  PokemonEnum2["Gogoat"] = "gogoat";
  PokemonEnum2["Golbat"] = "golbat";
  PokemonEnum2["Goldeen"] = "goldeen";
  PokemonEnum2["Golduck"] = "golduck";
  PokemonEnum2["Golem"] = "golem";
  PokemonEnum2["Golemalola"] = "golemalola";
  PokemonEnum2["Golett"] = "golett";
  PokemonEnum2["Golisopod"] = "golisopod";
  PokemonEnum2["Golurk"] = "golurk";
  PokemonEnum2["Goodra"] = "goodra";
  PokemonEnum2["Goodrahisui"] = "goodrahisui";
  PokemonEnum2["Goomy"] = "goomy";
  PokemonEnum2["Gorebyss"] = "gorebyss";
  PokemonEnum2["Gossifleur"] = "gossifleur";
  PokemonEnum2["Gothita"] = "gothita";
  PokemonEnum2["Gothitelle"] = "gothitelle";
  PokemonEnum2["Gothorita"] = "gothorita";
  PokemonEnum2["Gourgeist"] = "gourgeist";
  PokemonEnum2["Gourgeistlarge"] = "gourgeistlarge";
  PokemonEnum2["Gourgeistsmall"] = "gourgeistsmall";
  PokemonEnum2["Gourgeistsuper"] = "gourgeistsuper";
  PokemonEnum2["Granbull"] = "granbull";
  PokemonEnum2["Grapploct"] = "grapploct";
  PokemonEnum2["Graveler"] = "graveler";
  PokemonEnum2["Graveleralola"] = "graveleralola";
  PokemonEnum2["Greedent"] = "greedent";
  PokemonEnum2["Greninja"] = "greninja";
  PokemonEnum2["Greninjaash"] = "greninjaash";
  PokemonEnum2["Grimer"] = "grimer";
  PokemonEnum2["Grimeralola"] = "grimeralola";
  PokemonEnum2["Grimmsnarl"] = "grimmsnarl";
  PokemonEnum2["Grimmsnarlgmax"] = "grimmsnarlgmax";
  PokemonEnum2["Grookey"] = "grookey";
  PokemonEnum2["Grotle"] = "grotle";
  PokemonEnum2["Groudon"] = "groudon";
  PokemonEnum2["Groudonprimal"] = "groudonprimal";
  PokemonEnum2["Grovyle"] = "grovyle";
  PokemonEnum2["Growlithe"] = "growlithe";
  PokemonEnum2["Growlithehisui"] = "growlithehisui";
  PokemonEnum2["Grubbin"] = "grubbin";
  PokemonEnum2["Grumpig"] = "grumpig";
  PokemonEnum2["Gulpin"] = "gulpin";
  PokemonEnum2["Gumshoos"] = "gumshoos";
  PokemonEnum2["Gumshoostotem"] = "gumshoostotem";
  PokemonEnum2["Gurdurr"] = "gurdurr";
  PokemonEnum2["Guzzlord"] = "guzzlord";
  PokemonEnum2["Gyarados"] = "gyarados";
  PokemonEnum2["Gyaradosmega"] = "gyaradosmega";
  PokemonEnum2["Hakamoo"] = "hakamoo";
  PokemonEnum2["Happiny"] = "happiny";
  PokemonEnum2["Hariyama"] = "hariyama";
  PokemonEnum2["Hatenna"] = "hatenna";
  PokemonEnum2["Hatterene"] = "hatterene";
  PokemonEnum2["Hatterenegmax"] = "hatterenegmax";
  PokemonEnum2["Hattrem"] = "hattrem";
  PokemonEnum2["Haunter"] = "haunter";
  PokemonEnum2["Hawlucha"] = "hawlucha";
  PokemonEnum2["Haxorus"] = "haxorus";
  PokemonEnum2["Heatmor"] = "heatmor";
  PokemonEnum2["Heatran"] = "heatran";
  PokemonEnum2["Heliolisk"] = "heliolisk";
  PokemonEnum2["Helioptile"] = "helioptile";
  PokemonEnum2["Heracross"] = "heracross";
  PokemonEnum2["Heracrossmega"] = "heracrossmega";
  PokemonEnum2["Herdier"] = "herdier";
  PokemonEnum2["Hippopotas"] = "hippopotas";
  PokemonEnum2["Hippowdon"] = "hippowdon";
  PokemonEnum2["Hitmonchan"] = "hitmonchan";
  PokemonEnum2["Hitmonlee"] = "hitmonlee";
  PokemonEnum2["Hitmontop"] = "hitmontop";
  PokemonEnum2["Honchkrow"] = "honchkrow";
  PokemonEnum2["Honedge"] = "honedge";
  PokemonEnum2["Hooh"] = "hooh";
  PokemonEnum2["Hoopa"] = "hoopa";
  PokemonEnum2["Hoopaunbound"] = "hoopaunbound";
  PokemonEnum2["Hoothoot"] = "hoothoot";
  PokemonEnum2["Hoppip"] = "hoppip";
  PokemonEnum2["Horsea"] = "horsea";
  PokemonEnum2["Houndoom"] = "houndoom";
  PokemonEnum2["Houndoommega"] = "houndoommega";
  PokemonEnum2["Houndour"] = "houndour";
  PokemonEnum2["Huntail"] = "huntail";
  PokemonEnum2["Hydreigon"] = "hydreigon";
  PokemonEnum2["Hypno"] = "hypno";
  PokemonEnum2["Igglybuff"] = "igglybuff";
  PokemonEnum2["Illumise"] = "illumise";
  PokemonEnum2["Impidimp"] = "impidimp";
  PokemonEnum2["Incineroar"] = "incineroar";
  PokemonEnum2["Indeedee"] = "indeedee";
  PokemonEnum2["Indeedeef"] = "indeedeef";
  PokemonEnum2["Infernape"] = "infernape";
  PokemonEnum2["Inkay"] = "inkay";
  PokemonEnum2["Inteleon"] = "inteleon";
  PokemonEnum2["Inteleongmax"] = "inteleongmax";
  PokemonEnum2["Ivysaur"] = "ivysaur";
  PokemonEnum2["Jangmoo"] = "jangmoo";
  PokemonEnum2["Jellicent"] = "jellicent";
  PokemonEnum2["Jellicentfemale"] = "jellicentfemale";
  PokemonEnum2["Jigglypuff"] = "jigglypuff";
  PokemonEnum2["Jirachi"] = "jirachi";
  PokemonEnum2["Jolteon"] = "jolteon";
  PokemonEnum2["Joltik"] = "joltik";
  PokemonEnum2["Jumbao"] = "jumbao";
  PokemonEnum2["Jumpluff"] = "jumpluff";
  PokemonEnum2["Justyke"] = "justyke";
  PokemonEnum2["Jynx"] = "jynx";
  PokemonEnum2["Kabuto"] = "kabuto";
  PokemonEnum2["Kabutops"] = "kabutops";
  PokemonEnum2["Kadabra"] = "kadabra";
  PokemonEnum2["Kakuna"] = "kakuna";
  PokemonEnum2["Kangaskhan"] = "kangaskhan";
  PokemonEnum2["Kangaskhanmega"] = "kangaskhanmega";
  PokemonEnum2["Karrablast"] = "karrablast";
  PokemonEnum2["Kartana"] = "kartana";
  PokemonEnum2["Kecleon"] = "kecleon";
  PokemonEnum2["Keldeo"] = "keldeo";
  PokemonEnum2["Keldeoresolute"] = "keldeoresolute";
  PokemonEnum2["Kerfluffle"] = "kerfluffle";
  PokemonEnum2["Kingdra"] = "kingdra";
  PokemonEnum2["Kingler"] = "kingler";
  PokemonEnum2["Kinglergmax"] = "kinglergmax";
  PokemonEnum2["Kirlia"] = "kirlia";
  PokemonEnum2["Kitsunoh"] = "kitsunoh";
  PokemonEnum2["Klang"] = "klang";
  PokemonEnum2["Kleavor"] = "kleavor";
  PokemonEnum2["Klefki"] = "klefki";
  PokemonEnum2["Klink"] = "klink";
  PokemonEnum2["Klinklang"] = "klinklang";
  PokemonEnum2["Koffing"] = "koffing";
  PokemonEnum2["Komala"] = "komala";
  PokemonEnum2["Kommoo"] = "kommoo";
  PokemonEnum2["Kommoototem"] = "kommoototem";
  PokemonEnum2["Krabby"] = "krabby";
  PokemonEnum2["Kricketot"] = "kricketot";
  PokemonEnum2["Kricketune"] = "kricketune";
  PokemonEnum2["Krilowatt"] = "krilowatt";
  PokemonEnum2["Krokorok"] = "krokorok";
  PokemonEnum2["Krookodile"] = "krookodile";
  PokemonEnum2["Kubfu"] = "kubfu";
  PokemonEnum2["Kyogre"] = "kyogre";
  PokemonEnum2["Kyogreprimal"] = "kyogreprimal";
  PokemonEnum2["Kyurem"] = "kyurem";
  PokemonEnum2["Kyuremblack"] = "kyuremblack";
  PokemonEnum2["Kyuremwhite"] = "kyuremwhite";
  PokemonEnum2["Lairon"] = "lairon";
  PokemonEnum2["Lampent"] = "lampent";
  PokemonEnum2["Landorus"] = "landorus";
  PokemonEnum2["Landorustherian"] = "landorustherian";
  PokemonEnum2["Lanturn"] = "lanturn";
  PokemonEnum2["Lapras"] = "lapras";
  PokemonEnum2["Laprasgmax"] = "laprasgmax";
  PokemonEnum2["Larvesta"] = "larvesta";
  PokemonEnum2["Larvitar"] = "larvitar";
  PokemonEnum2["Latias"] = "latias";
  PokemonEnum2["Latiasmega"] = "latiasmega";
  PokemonEnum2["Latios"] = "latios";
  PokemonEnum2["Latiosmega"] = "latiosmega";
  PokemonEnum2["Leafeon"] = "leafeon";
  PokemonEnum2["Leavanny"] = "leavanny";
  PokemonEnum2["Ledian"] = "ledian";
  PokemonEnum2["Ledyba"] = "ledyba";
  PokemonEnum2["Lickilicky"] = "lickilicky";
  PokemonEnum2["Lickitung"] = "lickitung";
  PokemonEnum2["Liepard"] = "liepard";
  PokemonEnum2["Lileep"] = "lileep";
  PokemonEnum2["Lilligant"] = "lilligant";
  PokemonEnum2["Lilliganthisui"] = "lilliganthisui";
  PokemonEnum2["Lillipup"] = "lillipup";
  PokemonEnum2["Linoone"] = "linoone";
  PokemonEnum2["Linoonegalar"] = "linoonegalar";
  PokemonEnum2["Litleo"] = "litleo";
  PokemonEnum2["Litten"] = "litten";
  PokemonEnum2["Litwick"] = "litwick";
  PokemonEnum2["Lombre"] = "lombre";
  PokemonEnum2["Lopunny"] = "lopunny";
  PokemonEnum2["Lopunnymega"] = "lopunnymega";
  PokemonEnum2["Lotad"] = "lotad";
  PokemonEnum2["Loudred"] = "loudred";
  PokemonEnum2["Lucario"] = "lucario";
  PokemonEnum2["Lucariomega"] = "lucariomega";
  PokemonEnum2["Ludicolo"] = "ludicolo";
  PokemonEnum2["Lugia"] = "lugia";
  PokemonEnum2["Lumineon"] = "lumineon";
  PokemonEnum2["Lunala"] = "lunala";
  PokemonEnum2["Lunatone"] = "lunatone";
  PokemonEnum2["Lurantis"] = "lurantis";
  PokemonEnum2["Lurantistotem"] = "lurantistotem";
  PokemonEnum2["Luvdisc"] = "luvdisc";
  PokemonEnum2["Luxio"] = "luxio";
  PokemonEnum2["Luxray"] = "luxray";
  PokemonEnum2["Lycanroc"] = "lycanroc";
  PokemonEnum2["Lycanrocdusk"] = "lycanrocdusk";
  PokemonEnum2["Lycanrocmidnight"] = "lycanrocmidnight";
  PokemonEnum2["Machamp"] = "machamp";
  PokemonEnum2["Machampgmax"] = "machampgmax";
  PokemonEnum2["Machoke"] = "machoke";
  PokemonEnum2["Machop"] = "machop";
  PokemonEnum2["Magby"] = "magby";
  PokemonEnum2["Magcargo"] = "magcargo";
  PokemonEnum2["Magearna"] = "magearna";
  PokemonEnum2["Magearnaoriginal"] = "magearnaoriginal";
  PokemonEnum2["Magikarp"] = "magikarp";
  PokemonEnum2["Magmar"] = "magmar";
  PokemonEnum2["Magmortar"] = "magmortar";
  PokemonEnum2["Magnemite"] = "magnemite";
  PokemonEnum2["Magneton"] = "magneton";
  PokemonEnum2["Magnezone"] = "magnezone";
  PokemonEnum2["Makuhita"] = "makuhita";
  PokemonEnum2["Malaconda"] = "malaconda";
  PokemonEnum2["Malamar"] = "malamar";
  PokemonEnum2["Mamoswine"] = "mamoswine";
  PokemonEnum2["Manaphy"] = "manaphy";
  PokemonEnum2["Mandibuzz"] = "mandibuzz";
  PokemonEnum2["Manectric"] = "manectric";
  PokemonEnum2["Manectricmega"] = "manectricmega";
  PokemonEnum2["Mankey"] = "mankey";
  PokemonEnum2["Mantine"] = "mantine";
  PokemonEnum2["Mantyke"] = "mantyke";
  PokemonEnum2["Maractus"] = "maractus";
  PokemonEnum2["Mareanie"] = "mareanie";
  PokemonEnum2["Mareep"] = "mareep";
  PokemonEnum2["Marill"] = "marill";
  PokemonEnum2["Marowak"] = "marowak";
  PokemonEnum2["Marowakalola"] = "marowakalola";
  PokemonEnum2["Marowakalolatotem"] = "marowakalolatotem";
  PokemonEnum2["Marshadow"] = "marshadow";
  PokemonEnum2["Marshtomp"] = "marshtomp";
  PokemonEnum2["Masquerain"] = "masquerain";
  PokemonEnum2["Mawile"] = "mawile";
  PokemonEnum2["Mawilemega"] = "mawilemega";
  PokemonEnum2["Medicham"] = "medicham";
  PokemonEnum2["Medichammega"] = "medichammega";
  PokemonEnum2["Meditite"] = "meditite";
  PokemonEnum2["Meganium"] = "meganium";
  PokemonEnum2["Melmetal"] = "melmetal";
  PokemonEnum2["Melmetalgmax"] = "melmetalgmax";
  PokemonEnum2["Meloetta"] = "meloetta";
  PokemonEnum2["Meloettapirouette"] = "meloettapirouette";
  PokemonEnum2["Meltan"] = "meltan";
  PokemonEnum2["Meowstic"] = "meowstic";
  PokemonEnum2["Meowsticf"] = "meowsticf";
  PokemonEnum2["Meowth"] = "meowth";
  PokemonEnum2["Meowthalola"] = "meowthalola";
  PokemonEnum2["Meowthgalar"] = "meowthgalar";
  PokemonEnum2["Meowthgmax"] = "meowthgmax";
  PokemonEnum2["Mesprit"] = "mesprit";
  PokemonEnum2["Metagross"] = "metagross";
  PokemonEnum2["Metagrossmega"] = "metagrossmega";
  PokemonEnum2["Metang"] = "metang";
  PokemonEnum2["Metapod"] = "metapod";
  PokemonEnum2["Mew"] = "mew";
  PokemonEnum2["Mewtwo"] = "mewtwo";
  PokemonEnum2["Mewtwomegax"] = "mewtwomegax";
  PokemonEnum2["Mewtwomegay"] = "mewtwomegay";
  PokemonEnum2["Miasmaw"] = "miasmaw";
  PokemonEnum2["Miasmite"] = "miasmite";
  PokemonEnum2["Mienfoo"] = "mienfoo";
  PokemonEnum2["Mienshao"] = "mienshao";
  PokemonEnum2["Mightyena"] = "mightyena";
  PokemonEnum2["Milcery"] = "milcery";
  PokemonEnum2["Milotic"] = "milotic";
  PokemonEnum2["Miltank"] = "miltank";
  PokemonEnum2["Mimejr"] = "mimejr";
  PokemonEnum2["Mimikyu"] = "mimikyu";
  PokemonEnum2["Mimikyubusted"] = "mimikyubusted";
  PokemonEnum2["Mimikyubustedtotem"] = "mimikyubustedtotem";
  PokemonEnum2["Mimikyutotem"] = "mimikyutotem";
  PokemonEnum2["Minccino"] = "minccino";
  PokemonEnum2["Minior"] = "minior";
  PokemonEnum2["Miniormeteor"] = "miniormeteor";
  PokemonEnum2["Minun"] = "minun";
  PokemonEnum2["Misdreavus"] = "misdreavus";
  PokemonEnum2["Mismagius"] = "mismagius";
  PokemonEnum2["Missingno"] = "missingno";
  PokemonEnum2["Mollux"] = "mollux";
  PokemonEnum2["Moltres"] = "moltres";
  PokemonEnum2["Moltresgalar"] = "moltresgalar";
  PokemonEnum2["Monferno"] = "monferno";
  PokemonEnum2["Monohm"] = "monohm";
  PokemonEnum2["Morelull"] = "morelull";
  PokemonEnum2["Morgrem"] = "morgrem";
  PokemonEnum2["Morpeko"] = "morpeko";
  PokemonEnum2["Morpekohangry"] = "morpekohangry";
  PokemonEnum2["Mothim"] = "mothim";
  PokemonEnum2["Mrmime"] = "mrmime";
  PokemonEnum2["Mrmimegalar"] = "mrmimegalar";
  PokemonEnum2["Mrrime"] = "mrrime";
  PokemonEnum2["Mudbray"] = "mudbray";
  PokemonEnum2["Mudkip"] = "mudkip";
  PokemonEnum2["Mudsdale"] = "mudsdale";
  PokemonEnum2["Muk"] = "muk";
  PokemonEnum2["Mukalola"] = "mukalola";
  PokemonEnum2["Mumbao"] = "mumbao";
  PokemonEnum2["Munchlax"] = "munchlax";
  PokemonEnum2["Munna"] = "munna";
  PokemonEnum2["Murkrow"] = "murkrow";
  PokemonEnum2["Musharna"] = "musharna";
  PokemonEnum2["Naganadel"] = "naganadel";
  PokemonEnum2["Natu"] = "natu";
  PokemonEnum2["Naviathan"] = "naviathan";
  PokemonEnum2["Necrozma"] = "necrozma";
  PokemonEnum2["Necrozmadawnwings"] = "necrozmadawnwings";
  PokemonEnum2["Necrozmaduskmane"] = "necrozmaduskmane";
  PokemonEnum2["Necrozmaultra"] = "necrozmaultra";
  PokemonEnum2["Necturine"] = "necturine";
  PokemonEnum2["Necturna"] = "necturna";
  PokemonEnum2["Nickit"] = "nickit";
  PokemonEnum2["Nidoking"] = "nidoking";
  PokemonEnum2["Nidoqueen"] = "nidoqueen";
  PokemonEnum2["Nidoranf"] = "nidoranf";
  PokemonEnum2["Nidoranm"] = "nidoranm";
  PokemonEnum2["Nidorina"] = "nidorina";
  PokemonEnum2["Nidorino"] = "nidorino";
  PokemonEnum2["Nihilego"] = "nihilego";
  PokemonEnum2["Nincada"] = "nincada";
  PokemonEnum2["Ninetales"] = "ninetales";
  PokemonEnum2["Ninetalesalola"] = "ninetalesalola";
  PokemonEnum2["Ninjask"] = "ninjask";
  PokemonEnum2["Noctowl"] = "noctowl";
  PokemonEnum2["Nohface"] = "nohface";
  PokemonEnum2["Noibat"] = "noibat";
  PokemonEnum2["Noivern"] = "noivern";
  PokemonEnum2["Nosepass"] = "nosepass";
  PokemonEnum2["Numel"] = "numel";
  PokemonEnum2["Nuzleaf"] = "nuzleaf";
  PokemonEnum2["Obstagoon"] = "obstagoon";
  PokemonEnum2["Octillery"] = "octillery";
  PokemonEnum2["Oddish"] = "oddish";
  PokemonEnum2["Omanyte"] = "omanyte";
  PokemonEnum2["Omastar"] = "omastar";
  PokemonEnum2["Onix"] = "onix";
  PokemonEnum2["Oranguru"] = "oranguru";
  PokemonEnum2["Orbeetle"] = "orbeetle";
  PokemonEnum2["Orbeetlegmax"] = "orbeetlegmax";
  PokemonEnum2["Oricorio"] = "oricorio";
  PokemonEnum2["Oricoriopau"] = "oricoriopau";
  PokemonEnum2["Oricoriopompom"] = "oricoriopompom";
  PokemonEnum2["Oricoriosensu"] = "oricoriosensu";
  PokemonEnum2["Oshawott"] = "oshawott";
  PokemonEnum2["Overqwil"] = "overqwil";
  PokemonEnum2["Pachirisu"] = "pachirisu";
  PokemonEnum2["Pajantom"] = "pajantom";
  PokemonEnum2["Palkia"] = "palkia";
  PokemonEnum2["Palkiaorigin"] = "palkiaorigin";
  PokemonEnum2["Palossand"] = "palossand";
  PokemonEnum2["Palpitoad"] = "palpitoad";
  PokemonEnum2["Pancham"] = "pancham";
  PokemonEnum2["Pangoro"] = "pangoro";
  PokemonEnum2["Panpour"] = "panpour";
  PokemonEnum2["Pansage"] = "pansage";
  PokemonEnum2["Pansear"] = "pansear";
  PokemonEnum2["Paras"] = "paras";
  PokemonEnum2["Parasect"] = "parasect";
  PokemonEnum2["Passimian"] = "passimian";
  PokemonEnum2["Patrat"] = "patrat";
  PokemonEnum2["Pawniard"] = "pawniard";
  PokemonEnum2["Pelipper"] = "pelipper";
  PokemonEnum2["Perrserker"] = "perrserker";
  PokemonEnum2["Persian"] = "persian";
  PokemonEnum2["Persianalola"] = "persianalola";
  PokemonEnum2["Petilil"] = "petilil";
  PokemonEnum2["Phanpy"] = "phanpy";
  PokemonEnum2["Phantump"] = "phantump";
  PokemonEnum2["Pheromosa"] = "pheromosa";
  PokemonEnum2["Phione"] = "phione";
  PokemonEnum2["Pichu"] = "pichu";
  PokemonEnum2["Pichuspikyeared"] = "pichuspikyeared";
  PokemonEnum2["Pidgeot"] = "pidgeot";
  PokemonEnum2["Pidgeotmega"] = "pidgeotmega";
  PokemonEnum2["Pidgeotto"] = "pidgeotto";
  PokemonEnum2["Pidgey"] = "pidgey";
  PokemonEnum2["Pidove"] = "pidove";
  PokemonEnum2["Pignite"] = "pignite";
  PokemonEnum2["Pikachu"] = "pikachu";
  PokemonEnum2["Pikachualola"] = "pikachualola";
  PokemonEnum2["Pikachubelle"] = "pikachubelle";
  PokemonEnum2["Pikachucosplay"] = "pikachucosplay";
  PokemonEnum2["Pikachugmax"] = "pikachugmax";
  PokemonEnum2["Pikachuhoenn"] = "pikachuhoenn";
  PokemonEnum2["Pikachukalos"] = "pikachukalos";
  PokemonEnum2["Pikachulibre"] = "pikachulibre";
  PokemonEnum2["Pikachuoriginal"] = "pikachuoriginal";
  PokemonEnum2["Pikachupartner"] = "pikachupartner";
  PokemonEnum2["Pikachuphd"] = "pikachuphd";
  PokemonEnum2["Pikachupopstar"] = "pikachupopstar";
  PokemonEnum2["Pikachurockstar"] = "pikachurockstar";
  PokemonEnum2["Pikachusinnoh"] = "pikachusinnoh";
  PokemonEnum2["Pikachustarter"] = "pikachustarter";
  PokemonEnum2["Pikachuunova"] = "pikachuunova";
  PokemonEnum2["Pikachuworld"] = "pikachuworld";
  PokemonEnum2["Pikipek"] = "pikipek";
  PokemonEnum2["Piloswine"] = "piloswine";
  PokemonEnum2["Pincurchin"] = "pincurchin";
  PokemonEnum2["Pineco"] = "pineco";
  PokemonEnum2["Pinsir"] = "pinsir";
  PokemonEnum2["Pinsirmega"] = "pinsirmega";
  PokemonEnum2["Piplup"] = "piplup";
  PokemonEnum2["Plasmanta"] = "plasmanta";
  PokemonEnum2["Pluffle"] = "pluffle";
  PokemonEnum2["Plusle"] = "plusle";
  PokemonEnum2["Poipole"] = "poipole";
  PokemonEnum2["Pokestarblackbelt"] = "pokestarblackbelt";
  PokemonEnum2["Pokestarblackdoor"] = "pokestarblackdoor";
  PokemonEnum2["Pokestarbrycenman"] = "pokestarbrycenman";
  PokemonEnum2["Pokestarf00"] = "pokestarf00";
  PokemonEnum2["Pokestarf002"] = "pokestarf002";
  PokemonEnum2["Pokestargiant"] = "pokestargiant";
  PokemonEnum2["Pokestarhumanoid"] = "pokestarhumanoid";
  PokemonEnum2["Pokestarmonster"] = "pokestarmonster";
  PokemonEnum2["Pokestarmt"] = "pokestarmt";
  PokemonEnum2["Pokestarmt2"] = "pokestarmt2";
  PokemonEnum2["Pokestarsmeargle"] = "pokestarsmeargle";
  PokemonEnum2["Pokestarspirit"] = "pokestarspirit";
  PokemonEnum2["Pokestartransport"] = "pokestartransport";
  PokemonEnum2["Pokestarufo"] = "pokestarufo";
  PokemonEnum2["Pokestarufo2"] = "pokestarufo2";
  PokemonEnum2["Pokestarufopropu2"] = "pokestarufopropu2";
  PokemonEnum2["Pokestarwhitedoor"] = "pokestarwhitedoor";
  PokemonEnum2["Politoed"] = "politoed";
  PokemonEnum2["Poliwag"] = "poliwag";
  PokemonEnum2["Poliwhirl"] = "poliwhirl";
  PokemonEnum2["Poliwrath"] = "poliwrath";
  PokemonEnum2["Polteageist"] = "polteageist";
  PokemonEnum2["Polteageistantique"] = "polteageistantique";
  PokemonEnum2["Ponyta"] = "ponyta";
  PokemonEnum2["Ponytagalar"] = "ponytagalar";
  PokemonEnum2["Poochyena"] = "poochyena";
  PokemonEnum2["Popplio"] = "popplio";
  PokemonEnum2["Porygon"] = "porygon";
  PokemonEnum2["Porygon2"] = "porygon2";
  PokemonEnum2["Porygonz"] = "porygonz";
  PokemonEnum2["Primarina"] = "primarina";
  PokemonEnum2["Primeape"] = "primeape";
  PokemonEnum2["Prinplup"] = "prinplup";
  PokemonEnum2["Privatyke"] = "privatyke";
  PokemonEnum2["Probopass"] = "probopass";
  PokemonEnum2["Protowatt"] = "protowatt";
  PokemonEnum2["Psyduck"] = "psyduck";
  PokemonEnum2["Pumpkaboo"] = "pumpkaboo";
  PokemonEnum2["Pumpkaboolarge"] = "pumpkaboolarge";
  PokemonEnum2["Pumpkaboosmall"] = "pumpkaboosmall";
  PokemonEnum2["Pumpkaboosuper"] = "pumpkaboosuper";
  PokemonEnum2["Pupitar"] = "pupitar";
  PokemonEnum2["Purrloin"] = "purrloin";
  PokemonEnum2["Purugly"] = "purugly";
  PokemonEnum2["Pyroak"] = "pyroak";
  PokemonEnum2["Pyroar"] = "pyroar";
  PokemonEnum2["Pyukumuku"] = "pyukumuku";
  PokemonEnum2["Quagsire"] = "quagsire";
  PokemonEnum2["Quilava"] = "quilava";
  PokemonEnum2["Quilladin"] = "quilladin";
  PokemonEnum2["Qwilfish"] = "qwilfish";
  PokemonEnum2["Qwilfishhisui"] = "qwilfishhisui";
  PokemonEnum2["Raboot"] = "raboot";
  PokemonEnum2["Raichu"] = "raichu";
  PokemonEnum2["Raichualola"] = "raichualola";
  PokemonEnum2["Raikou"] = "raikou";
  PokemonEnum2["Ralts"] = "ralts";
  PokemonEnum2["Rampardos"] = "rampardos";
  PokemonEnum2["Rapidash"] = "rapidash";
  PokemonEnum2["Rapidashgalar"] = "rapidashgalar";
  PokemonEnum2["Raticate"] = "raticate";
  PokemonEnum2["Raticatealola"] = "raticatealola";
  PokemonEnum2["Raticatealolatotem"] = "raticatealolatotem";
  PokemonEnum2["Rattata"] = "rattata";
  PokemonEnum2["Rattataalola"] = "rattataalola";
  PokemonEnum2["Rayquaza"] = "rayquaza";
  PokemonEnum2["Rayquazamega"] = "rayquazamega";
  PokemonEnum2["Rebble"] = "rebble";
  PokemonEnum2["Regice"] = "regice";
  PokemonEnum2["Regidrago"] = "regidrago";
  PokemonEnum2["Regieleki"] = "regieleki";
  PokemonEnum2["Regigigas"] = "regigigas";
  PokemonEnum2["Regirock"] = "regirock";
  PokemonEnum2["Registeel"] = "registeel";
  PokemonEnum2["Relicanth"] = "relicanth";
  PokemonEnum2["Remoraid"] = "remoraid";
  PokemonEnum2["Reshiram"] = "reshiram";
  PokemonEnum2["Reuniclus"] = "reuniclus";
  PokemonEnum2["Revenankh"] = "revenankh";
  PokemonEnum2["Rhydon"] = "rhydon";
  PokemonEnum2["Rhyhorn"] = "rhyhorn";
  PokemonEnum2["Rhyperior"] = "rhyperior";
  PokemonEnum2["Ribombee"] = "ribombee";
  PokemonEnum2["Ribombeetotem"] = "ribombeetotem";
  PokemonEnum2["Rillaboom"] = "rillaboom";
  PokemonEnum2["Rillaboomgmax"] = "rillaboomgmax";
  PokemonEnum2["Riolu"] = "riolu";
  PokemonEnum2["Rockruff"] = "rockruff";
  PokemonEnum2["Roggenrola"] = "roggenrola";
  PokemonEnum2["Rolycoly"] = "rolycoly";
  PokemonEnum2["Rookidee"] = "rookidee";
  PokemonEnum2["Roselia"] = "roselia";
  PokemonEnum2["Roserade"] = "roserade";
  PokemonEnum2["Rotom"] = "rotom";
  PokemonEnum2["Rotomfan"] = "rotomfan";
  PokemonEnum2["Rotomfrost"] = "rotomfrost";
  PokemonEnum2["Rotomheat"] = "rotomheat";
  PokemonEnum2["Rotommow"] = "rotommow";
  PokemonEnum2["Rotomwash"] = "rotomwash";
  PokemonEnum2["Rowlet"] = "rowlet";
  PokemonEnum2["Rufflet"] = "rufflet";
  PokemonEnum2["Runerigus"] = "runerigus";
  PokemonEnum2["Sableye"] = "sableye";
  PokemonEnum2["Sableyemega"] = "sableyemega";
  PokemonEnum2["Saharaja"] = "saharaja";
  PokemonEnum2["Salamence"] = "salamence";
  PokemonEnum2["Salamencemega"] = "salamencemega";
  PokemonEnum2["Salandit"] = "salandit";
  PokemonEnum2["Salazzle"] = "salazzle";
  PokemonEnum2["Salazzletotem"] = "salazzletotem";
  PokemonEnum2["Samurott"] = "samurott";
  PokemonEnum2["Samurotthisui"] = "samurotthisui";
  PokemonEnum2["Sandaconda"] = "sandaconda";
  PokemonEnum2["Sandacondagmax"] = "sandacondagmax";
  PokemonEnum2["Sandile"] = "sandile";
  PokemonEnum2["Sandshrew"] = "sandshrew";
  PokemonEnum2["Sandshrewalola"] = "sandshrewalola";
  PokemonEnum2["Sandslash"] = "sandslash";
  PokemonEnum2["Sandslashalola"] = "sandslashalola";
  PokemonEnum2["Sandygast"] = "sandygast";
  PokemonEnum2["Sawk"] = "sawk";
  PokemonEnum2["Sawsbuck"] = "sawsbuck";
  PokemonEnum2["Scatterbug"] = "scatterbug";
  PokemonEnum2["Sceptile"] = "sceptile";
  PokemonEnum2["Sceptilemega"] = "sceptilemega";
  PokemonEnum2["Scizor"] = "scizor";
  PokemonEnum2["Scizormega"] = "scizormega";
  PokemonEnum2["Scolipede"] = "scolipede";
  PokemonEnum2["Scorbunny"] = "scorbunny";
  PokemonEnum2["Scrafty"] = "scrafty";
  PokemonEnum2["Scraggy"] = "scraggy";
  PokemonEnum2["Scratchet"] = "scratchet";
  PokemonEnum2["Scyther"] = "scyther";
  PokemonEnum2["Seadra"] = "seadra";
  PokemonEnum2["Seaking"] = "seaking";
  PokemonEnum2["Sealeo"] = "sealeo";
  PokemonEnum2["Seedot"] = "seedot";
  PokemonEnum2["Seel"] = "seel";
  PokemonEnum2["Seismitoad"] = "seismitoad";
  PokemonEnum2["Sentret"] = "sentret";
  PokemonEnum2["Serperior"] = "serperior";
  PokemonEnum2["Servine"] = "servine";
  PokemonEnum2["Seviper"] = "seviper";
  PokemonEnum2["Sewaddle"] = "sewaddle";
  PokemonEnum2["Sharpedo"] = "sharpedo";
  PokemonEnum2["Sharpedomega"] = "sharpedomega";
  PokemonEnum2["Shaymin"] = "shaymin";
  PokemonEnum2["Shayminsky"] = "shayminsky";
  PokemonEnum2["Shedinja"] = "shedinja";
  PokemonEnum2["Shelgon"] = "shelgon";
  PokemonEnum2["Shellder"] = "shellder";
  PokemonEnum2["Shellos"] = "shellos";
  PokemonEnum2["Shelmet"] = "shelmet";
  PokemonEnum2["Shieldon"] = "shieldon";
  PokemonEnum2["Shiftry"] = "shiftry";
  PokemonEnum2["Shiinotic"] = "shiinotic";
  PokemonEnum2["Shinx"] = "shinx";
  PokemonEnum2["Shroomish"] = "shroomish";
  PokemonEnum2["Shuckle"] = "shuckle";
  PokemonEnum2["Shuppet"] = "shuppet";
  PokemonEnum2["Sigilyph"] = "sigilyph";
  PokemonEnum2["Silcoon"] = "silcoon";
  PokemonEnum2["Silicobra"] = "silicobra";
  PokemonEnum2["Silvally"] = "silvally";
  PokemonEnum2["Silvallybug"] = "silvallybug";
  PokemonEnum2["Silvallydark"] = "silvallydark";
  PokemonEnum2["Silvallydragon"] = "silvallydragon";
  PokemonEnum2["Silvallyelectric"] = "silvallyelectric";
  PokemonEnum2["Silvallyfairy"] = "silvallyfairy";
  PokemonEnum2["Silvallyfighting"] = "silvallyfighting";
  PokemonEnum2["Silvallyfire"] = "silvallyfire";
  PokemonEnum2["Silvallyflying"] = "silvallyflying";
  PokemonEnum2["Silvallyghost"] = "silvallyghost";
  PokemonEnum2["Silvallygrass"] = "silvallygrass";
  PokemonEnum2["Silvallyground"] = "silvallyground";
  PokemonEnum2["Silvallyice"] = "silvallyice";
  PokemonEnum2["Silvallypoison"] = "silvallypoison";
  PokemonEnum2["Silvallypsychic"] = "silvallypsychic";
  PokemonEnum2["Silvallyrock"] = "silvallyrock";
  PokemonEnum2["Silvallysteel"] = "silvallysteel";
  PokemonEnum2["Silvallywater"] = "silvallywater";
  PokemonEnum2["Simipour"] = "simipour";
  PokemonEnum2["Simisage"] = "simisage";
  PokemonEnum2["Simisear"] = "simisear";
  PokemonEnum2["Sinistea"] = "sinistea";
  PokemonEnum2["Sinisteaantique"] = "sinisteaantique";
  PokemonEnum2["Sirfetchd"] = "sirfetchd";
  PokemonEnum2["Sizzlipede"] = "sizzlipede";
  PokemonEnum2["Skarmory"] = "skarmory";
  PokemonEnum2["Skiddo"] = "skiddo";
  PokemonEnum2["Skiploom"] = "skiploom";
  PokemonEnum2["Skitty"] = "skitty";
  PokemonEnum2["Skorupi"] = "skorupi";
  PokemonEnum2["Skrelp"] = "skrelp";
  PokemonEnum2["Skuntank"] = "skuntank";
  PokemonEnum2["Skwovet"] = "skwovet";
  PokemonEnum2["Slaking"] = "slaking";
  PokemonEnum2["Slakoth"] = "slakoth";
  PokemonEnum2["Sliggoo"] = "sliggoo";
  PokemonEnum2["Sliggoohisui"] = "sliggoohisui";
  PokemonEnum2["Slowbro"] = "slowbro";
  PokemonEnum2["Slowbrogalar"] = "slowbrogalar";
  PokemonEnum2["Slowbromega"] = "slowbromega";
  PokemonEnum2["Slowking"] = "slowking";
  PokemonEnum2["Slowkinggalar"] = "slowkinggalar";
  PokemonEnum2["Slowpoke"] = "slowpoke";
  PokemonEnum2["Slowpokegalar"] = "slowpokegalar";
  PokemonEnum2["Slugma"] = "slugma";
  PokemonEnum2["Slurpuff"] = "slurpuff";
  PokemonEnum2["Smeargle"] = "smeargle";
  PokemonEnum2["Smogecko"] = "smogecko";
  PokemonEnum2["Smoguana"] = "smoguana";
  PokemonEnum2["Smokomodo"] = "smokomodo";
  PokemonEnum2["Smoochum"] = "smoochum";
  PokemonEnum2["Snaelstrom"] = "snaelstrom";
  PokemonEnum2["Sneasel"] = "sneasel";
  PokemonEnum2["Sneaselhisui"] = "sneaselhisui";
  PokemonEnum2["Sneasler"] = "sneasler";
  PokemonEnum2["Snivy"] = "snivy";
  PokemonEnum2["Snom"] = "snom";
  PokemonEnum2["Snorlax"] = "snorlax";
  PokemonEnum2["Snorlaxgmax"] = "snorlaxgmax";
  PokemonEnum2["Snorunt"] = "snorunt";
  PokemonEnum2["Snover"] = "snover";
  PokemonEnum2["Snubbull"] = "snubbull";
  PokemonEnum2["Snugglow"] = "snugglow";
  PokemonEnum2["Sobble"] = "sobble";
  PokemonEnum2["Solgaleo"] = "solgaleo";
  PokemonEnum2["Solosis"] = "solosis";
  PokemonEnum2["Solotl"] = "solotl";
  PokemonEnum2["Solrock"] = "solrock";
  PokemonEnum2["Spearow"] = "spearow";
  PokemonEnum2["Spectrier"] = "spectrier";
  PokemonEnum2["Spewpa"] = "spewpa";
  PokemonEnum2["Spheal"] = "spheal";
  PokemonEnum2["Spinarak"] = "spinarak";
  PokemonEnum2["Spinda"] = "spinda";
  PokemonEnum2["Spiritomb"] = "spiritomb";
  PokemonEnum2["Spoink"] = "spoink";
  PokemonEnum2["Spritzee"] = "spritzee";
  PokemonEnum2["Squirtle"] = "squirtle";
  PokemonEnum2["Stakataka"] = "stakataka";
  PokemonEnum2["Stantler"] = "stantler";
  PokemonEnum2["Staraptor"] = "staraptor";
  PokemonEnum2["Staravia"] = "staravia";
  PokemonEnum2["Starly"] = "starly";
  PokemonEnum2["Starmie"] = "starmie";
  PokemonEnum2["Staryu"] = "staryu";
  PokemonEnum2["Steelix"] = "steelix";
  PokemonEnum2["Steelixmega"] = "steelixmega";
  PokemonEnum2["Steenee"] = "steenee";
  PokemonEnum2["Stonjourner"] = "stonjourner";
  PokemonEnum2["Stoutland"] = "stoutland";
  PokemonEnum2["Stratagem"] = "stratagem";
  PokemonEnum2["Stufful"] = "stufful";
  PokemonEnum2["Stunfisk"] = "stunfisk";
  PokemonEnum2["Stunfiskgalar"] = "stunfiskgalar";
  PokemonEnum2["Stunky"] = "stunky";
  PokemonEnum2["Sudowoodo"] = "sudowoodo";
  PokemonEnum2["Suicune"] = "suicune";
  PokemonEnum2["Sunflora"] = "sunflora";
  PokemonEnum2["Sunkern"] = "sunkern";
  PokemonEnum2["Surskit"] = "surskit";
  PokemonEnum2["Swablu"] = "swablu";
  PokemonEnum2["Swadloon"] = "swadloon";
  PokemonEnum2["Swalot"] = "swalot";
  PokemonEnum2["Swampert"] = "swampert";
  PokemonEnum2["Swampertmega"] = "swampertmega";
  PokemonEnum2["Swanna"] = "swanna";
  PokemonEnum2["Swellow"] = "swellow";
  PokemonEnum2["Swinub"] = "swinub";
  PokemonEnum2["Swirlix"] = "swirlix";
  PokemonEnum2["Swirlpool"] = "swirlpool";
  PokemonEnum2["Swoobat"] = "swoobat";
  PokemonEnum2["Syclant"] = "syclant";
  PokemonEnum2["Syclar"] = "syclar";
  PokemonEnum2["Sylveon"] = "sylveon";
  PokemonEnum2["Tactite"] = "tactite";
  PokemonEnum2["Taillow"] = "taillow";
  PokemonEnum2["Talonflame"] = "talonflame";
  PokemonEnum2["Tangela"] = "tangela";
  PokemonEnum2["Tangrowth"] = "tangrowth";
  PokemonEnum2["Tapubulu"] = "tapubulu";
  PokemonEnum2["Tapufini"] = "tapufini";
  PokemonEnum2["Tapukoko"] = "tapukoko";
  PokemonEnum2["Tapulele"] = "tapulele";
  PokemonEnum2["Tauros"] = "tauros";
  PokemonEnum2["Teddiursa"] = "teddiursa";
  PokemonEnum2["Tentacool"] = "tentacool";
  PokemonEnum2["Tentacruel"] = "tentacruel";
  PokemonEnum2["Tepig"] = "tepig";
  PokemonEnum2["Terrakion"] = "terrakion";
  PokemonEnum2["Thievul"] = "thievul";
  PokemonEnum2["Throh"] = "throh";
  PokemonEnum2["Thundurus"] = "thundurus";
  PokemonEnum2["Thundurustherian"] = "thundurustherian";
  PokemonEnum2["Thwackey"] = "thwackey";
  PokemonEnum2["Timburr"] = "timburr";
  PokemonEnum2["Tirtouga"] = "tirtouga";
  PokemonEnum2["Togedemaru"] = "togedemaru";
  PokemonEnum2["Togedemarutotem"] = "togedemarutotem";
  PokemonEnum2["Togekiss"] = "togekiss";
  PokemonEnum2["Togepi"] = "togepi";
  PokemonEnum2["Togetic"] = "togetic";
  PokemonEnum2["Tomohawk"] = "tomohawk";
  PokemonEnum2["Torchic"] = "torchic";
  PokemonEnum2["Torkoal"] = "torkoal";
  PokemonEnum2["Tornadus"] = "tornadus";
  PokemonEnum2["Tornadustherian"] = "tornadustherian";
  PokemonEnum2["Torracat"] = "torracat";
  PokemonEnum2["Torterra"] = "torterra";
  PokemonEnum2["Totodile"] = "totodile";
  PokemonEnum2["Toucannon"] = "toucannon";
  PokemonEnum2["Toxapex"] = "toxapex";
  PokemonEnum2["Toxel"] = "toxel";
  PokemonEnum2["Toxicroak"] = "toxicroak";
  PokemonEnum2["Toxtricity"] = "toxtricity";
  PokemonEnum2["Toxtricitygmax"] = "toxtricitygmax";
  PokemonEnum2["Toxtricitylowkey"] = "toxtricitylowkey";
  PokemonEnum2["Toxtricitylowkeygmax"] = "toxtricitylowkeygmax";
  PokemonEnum2["Tranquill"] = "tranquill";
  PokemonEnum2["Trapinch"] = "trapinch";
  PokemonEnum2["Treecko"] = "treecko";
  PokemonEnum2["Trevenant"] = "trevenant";
  PokemonEnum2["Tropius"] = "tropius";
  PokemonEnum2["Trubbish"] = "trubbish";
  PokemonEnum2["Trumbeak"] = "trumbeak";
  PokemonEnum2["Tsareena"] = "tsareena";
  PokemonEnum2["Turtonator"] = "turtonator";
  PokemonEnum2["Turtwig"] = "turtwig";
  PokemonEnum2["Tympole"] = "tympole";
  PokemonEnum2["Tynamo"] = "tynamo";
  PokemonEnum2["Typenull"] = "typenull";
  PokemonEnum2["Typhlosion"] = "typhlosion";
  PokemonEnum2["Typhlosionhisui"] = "typhlosionhisui";
  PokemonEnum2["Tyranitar"] = "tyranitar";
  PokemonEnum2["Tyranitarmega"] = "tyranitarmega";
  PokemonEnum2["Tyrantrum"] = "tyrantrum";
  PokemonEnum2["Tyrogue"] = "tyrogue";
  PokemonEnum2["Tyrunt"] = "tyrunt";
  PokemonEnum2["Umbreon"] = "umbreon";
  PokemonEnum2["Unfezant"] = "unfezant";
  PokemonEnum2["Unown"] = "unown";
  PokemonEnum2["Ursaluna"] = "ursaluna";
  PokemonEnum2["Ursaring"] = "ursaring";
  PokemonEnum2["Urshifu"] = "urshifu";
  PokemonEnum2["Urshifugmax"] = "urshifugmax";
  PokemonEnum2["Urshifurapidstrike"] = "urshifurapidstrike";
  PokemonEnum2["Urshifurapidstrikegmax"] = "urshifurapidstrikegmax";
  PokemonEnum2["Uxie"] = "uxie";
  PokemonEnum2["Vanillish"] = "vanillish";
  PokemonEnum2["Vanillite"] = "vanillite";
  PokemonEnum2["Vanilluxe"] = "vanilluxe";
  PokemonEnum2["Vaporeon"] = "vaporeon";
  PokemonEnum2["Venipede"] = "venipede";
  PokemonEnum2["Venomicon"] = "venomicon";
  PokemonEnum2["Venomiconepilogue"] = "venomiconepilogue";
  PokemonEnum2["Venomoth"] = "venomoth";
  PokemonEnum2["Venonat"] = "venonat";
  PokemonEnum2["Venusaur"] = "venusaur";
  PokemonEnum2["Venusaurgmax"] = "venusaurgmax";
  PokemonEnum2["Venusaurmega"] = "venusaurmega";
  PokemonEnum2["Vespiquen"] = "vespiquen";
  PokemonEnum2["Vibrava"] = "vibrava";
  PokemonEnum2["Victini"] = "victini";
  PokemonEnum2["Victreebel"] = "victreebel";
  PokemonEnum2["Vigoroth"] = "vigoroth";
  PokemonEnum2["Vikavolt"] = "vikavolt";
  PokemonEnum2["Vikavolttotem"] = "vikavolttotem";
  PokemonEnum2["Vileplume"] = "vileplume";
  PokemonEnum2["Virizion"] = "virizion";
  PokemonEnum2["Vivillon"] = "vivillon";
  PokemonEnum2["Vivillonfancy"] = "vivillonfancy";
  PokemonEnum2["Vivillonpokeball"] = "vivillonpokeball";
  PokemonEnum2["Volbeat"] = "volbeat";
  PokemonEnum2["Volcanion"] = "volcanion";
  PokemonEnum2["Volcarona"] = "volcarona";
  PokemonEnum2["Volkraken"] = "volkraken";
  PokemonEnum2["Volkritter"] = "volkritter";
  PokemonEnum2["Voltorb"] = "voltorb";
  PokemonEnum2["Voltorbhisui"] = "voltorbhisui";
  PokemonEnum2["Voodoll"] = "voodoll";
  PokemonEnum2["Voodoom"] = "voodoom";
  PokemonEnum2["Vullaby"] = "vullaby";
  PokemonEnum2["Vulpix"] = "vulpix";
  PokemonEnum2["Vulpixalola"] = "vulpixalola";
  PokemonEnum2["Wailmer"] = "wailmer";
  PokemonEnum2["Wailord"] = "wailord";
  PokemonEnum2["Walrein"] = "walrein";
  PokemonEnum2["Wartortle"] = "wartortle";
  PokemonEnum2["Watchog"] = "watchog";
  PokemonEnum2["Weavile"] = "weavile";
  PokemonEnum2["Weedle"] = "weedle";
  PokemonEnum2["Weepinbell"] = "weepinbell";
  PokemonEnum2["Weezing"] = "weezing";
  PokemonEnum2["Weezinggalar"] = "weezinggalar";
  PokemonEnum2["Whimsicott"] = "whimsicott";
  PokemonEnum2["Whirlipede"] = "whirlipede";
  PokemonEnum2["Whiscash"] = "whiscash";
  PokemonEnum2["Whismur"] = "whismur";
  PokemonEnum2["Wigglytuff"] = "wigglytuff";
  PokemonEnum2["Wimpod"] = "wimpod";
  PokemonEnum2["Wingull"] = "wingull";
  PokemonEnum2["Wishiwashi"] = "wishiwashi";
  PokemonEnum2["Wishiwashischool"] = "wishiwashischool";
  PokemonEnum2["Wobbuffet"] = "wobbuffet";
  PokemonEnum2["Woobat"] = "woobat";
  PokemonEnum2["Wooloo"] = "wooloo";
  PokemonEnum2["Wooper"] = "wooper";
  PokemonEnum2["Wormadam"] = "wormadam";
  PokemonEnum2["Wormadamsandy"] = "wormadamsandy";
  PokemonEnum2["Wormadamtrash"] = "wormadamtrash";
  PokemonEnum2["Wurmple"] = "wurmple";
  PokemonEnum2["Wynaut"] = "wynaut";
  PokemonEnum2["Wyrdeer"] = "wyrdeer";
  PokemonEnum2["Xatu"] = "xatu";
  PokemonEnum2["Xerneas"] = "xerneas";
  PokemonEnum2["Xerneasneutral"] = "xerneasneutral";
  PokemonEnum2["Xurkitree"] = "xurkitree";
  PokemonEnum2["Yamask"] = "yamask";
  PokemonEnum2["Yamaskgalar"] = "yamaskgalar";
  PokemonEnum2["Yamper"] = "yamper";
  PokemonEnum2["Yanma"] = "yanma";
  PokemonEnum2["Yanmega"] = "yanmega";
  PokemonEnum2["Yungoos"] = "yungoos";
  PokemonEnum2["Yveltal"] = "yveltal";
  PokemonEnum2["Zacian"] = "zacian";
  PokemonEnum2["Zaciancrowned"] = "zaciancrowned";
  PokemonEnum2["Zamazenta"] = "zamazenta";
  PokemonEnum2["Zamazentacrowned"] = "zamazentacrowned";
  PokemonEnum2["Zangoose"] = "zangoose";
  PokemonEnum2["Zapdos"] = "zapdos";
  PokemonEnum2["Zapdosgalar"] = "zapdosgalar";
  PokemonEnum2["Zarude"] = "zarude";
  PokemonEnum2["Zarudedada"] = "zarudedada";
  PokemonEnum2["Zebstrika"] = "zebstrika";
  PokemonEnum2["Zekrom"] = "zekrom";
  PokemonEnum2["Zeraora"] = "zeraora";
  PokemonEnum2["Zigzagoon"] = "zigzagoon";
  PokemonEnum2["Zigzagoongalar"] = "zigzagoongalar";
  PokemonEnum2["Zoroark"] = "zoroark";
  PokemonEnum2["Zoroarkhisui"] = "zoroarkhisui";
  PokemonEnum2["Zorua"] = "zorua";
  PokemonEnum2["Zoruahisui"] = "zoruahisui";
  PokemonEnum2["Zubat"] = "zubat";
  PokemonEnum2["Zweilous"] = "zweilous";
  PokemonEnum2["Zygarde"] = "zygarde";
  PokemonEnum2["Zygarde10"] = "zygarde10";
  PokemonEnum2["Zygardecomplete"] = "zygardecomplete";
  return PokemonEnum2;
})(PokemonEnum || {});
var TypesEnum = /* @__PURE__ */ ((TypesEnum2) => {
  TypesEnum2["Bug"] = "bug";
  TypesEnum2["Dark"] = "dark";
  TypesEnum2["Dragon"] = "dragon";
  TypesEnum2["Electric"] = "electric";
  TypesEnum2["Fairy"] = "fairy";
  TypesEnum2["Fighting"] = "fighting";
  TypesEnum2["Fire"] = "fire";
  TypesEnum2["Flying"] = "flying";
  TypesEnum2["Ghost"] = "ghost";
  TypesEnum2["Grass"] = "grass";
  TypesEnum2["Ground"] = "ground";
  TypesEnum2["Ice"] = "ice";
  TypesEnum2["Normal"] = "normal";
  TypesEnum2["Poison"] = "poison";
  TypesEnum2["Psychic"] = "psychic";
  TypesEnum2["Rock"] = "rock";
  TypesEnum2["Steel"] = "steel";
  TypesEnum2["Water"] = "water";
  return TypesEnum2;
})(TypesEnum || {});
// Annotate the CommonJS export names for ESM import in node:
0 && (module.exports = {
  AbilitiesEnum,
  ItemsEnum,
  MovesEnum,
  PokemonEnum,
  TypesEnum
});
